<!DOCTYPE html>
<html>

<head>
<meta charset="UTF-8">
<title>Benchmark report</title>
<style type="text/css">
a { text-decoration: none; color: black }
a:hover, a:active { text-decoration: underline }
dt { float:left; clear: left; text-align: right; width: 13em }
dt, dt a { color: blue }
dt:after { content: ":" }
dd { margin-left: 13em; padding-left: 0.5em }
table { border-collapse: separate; border-spacing: 0px 0px }
thead, tfoot { text-align: center; font-weight: bold }
thead th { border-bottom: 2px solid black }
tfoot td { border-top: 3px solid black }
h3 { margin-top: 0 }
h4 { margin-bottom: 0 }
.query td { border-top: 1px solid black; vertical-align: top }
.sep { height: 3em }
.sep td { border-bottom: 1px solid black }
.sep-branch { height: 1em }
.row td { border-top: 1px solid black }
.obsolete { text-decoration: line-through }
.topic td, .topic { background-color: rgba(222,184,135,0.5) } /* BurlyWood */
.topic:hover td, .topic:hover { background-color: rgba(222,184,135,1) } /* BurlyWood */
.operation td, .operation { background-color: rgba(211,211,211,0.5) } /* LightGray */
.operation:hover td, .operation:hover { background-color: rgba(211,211,211,1) } /* LightGray */
.data td, .data { background-color: rgba(135,206,235,0.5) } /* SkyBlue */
.data:hover td, .data:hover { background-color: rgba(135,206,235,1) } /* SkyBlue */
.format td, .format { background-color: rgba(216,191,216,0.5) } /* Thistle */
.format:hover td, .format:hover { background-color: rgba(216,191,216,1) } /* Thistle */
.type { text-align: center; color: #666 }
.score { text-align: right; font-weight: bold }
.good { color: green }
.medium { color: yellow }
.bad { color: red }
.exact { text-decoration: underline }
tr .tp, tr:hover .tp { background-color: green }
tr .tp a { color: white }
tr .fp, tr:hover .fp { background-color: yellow }
tr .fn, tr:hover .fn { background-color: red }
</style>
</head>

<body>
<h1>Benchmark report</h1>
<p>2016-04-19T13:39:03+0300</p>

<h2>Input/Output</h2>
<dl>
<dt><a href="https://github.com/edamontology/edamontology/tree/master/releases">Ontology file</a></dt>
<dd>EDAM_1.14.owl</dd>
<dt>Query file</dt>
<dd>msutils.csv</dd>
<dt>Type</dt>
<dd>msutils</dd>
<dt>Output file</dt>
<dd>msutils.txt</dd>
<dt>Report file</dt>
<dd>&nbsp;</dd>
<dt>Benchmark report file</dt>
<dd>msutils_benchmark.html</dd>
</dl>

<h2>Preprocessing</h2>
<dl>
<dt>Remove freestanding numbers</dt>
<dd>false</dd>
<dt>Stopword list</dt>
<dd>lucene</dd>
<dt>Do stemming</dt>
<dd>true</dd>
<dt>Remove tokens of length</dt>
<dd>0</dd>
</dl>

<h2>Mapping</h2>
<dl>
<dt>Top matches per branch</dt>
<dd>3</dd>
<dt><a href="http://edamontology.org/page#Scope">Branches</a></dt>
<dd>[topic, operation, format]</dd>
<dt>Include obsolete concepts</dt>
<dd>true</dd>
</dl>

<h2>Mapping algorithm</h2>
<dl>
<dt>Compound words</dt>
<dd>1</dd>
<dt>Mismatch multiplier</dt>
<dd>2.0</dd>
<dt>Position off by 1</dt>
<dd>0.5</dd>
<dt>Position off by 2</dt>
<dd>0.125</dd>
<dt>Position loss</dt>
<dd>0.5</dd>
<dt>Concept weight</dt>
<dd>1.0</dd>
<dt>Query weight</dt>
<dd>1.0</dd>
<dt>Label multiplier</dt>
<dd>1.0</dd>
<dt>Exact synonym multiplier</dt>
<dd>0.95</dd>
<dt>Narrow/Broad multiplier</dt>
<dd>0.9</dd>
<dt>Definition multiplier</dt>
<dd>0.5</dd>
<dt>Comment multiplier</dt>
<dd>0.5</dd>
</dl>

<h2>Total</h2>
<dl>
<dt>Total EDAM concepts</dt>
<dd>3218</dd>
<dt>Total query keywords</dt>
<dd>236</dd>
<dt>Total mapper answers</dt>
<dd>236</dd>
<dt>Total matches</dt>
<dd>[1087, 791, 310] (2794)</dd><dt>Total query matches</dt>
<dd>[444, 259, 130] (833)</dd><dt>Total mapper matches</dt>
<dd>[708, 597, 213] (2124)</dd><dt><a href="https://en.wikipedia.org/wiki/Type_I_and_type_II_errors">Total mistakes</a></dt>
<dd>[1022, 726, 277] (2631)</dd><dt><a href="https://en.wikipedia.org/wiki/Type_I_and_type_II_errors">Total TP</a></dt>
<dd>[65, 65, 33] (163)</dd><dt><a href="https://en.wikipedia.org/wiki/Type_I_and_type_II_errors">Total FP</a></dt>
<dd>[643, 532, 180] (1961)</dd><dt><a href="https://en.wikipedia.org/wiki/Type_I_and_type_II_errors">Total FN</a></dt>
<dd>[379, 194, 97] (670)</dd></dl>

<h2>Mean</h2>
<dl>
<dt><a href="https://en.wikipedia.org/wiki/Precision_and_recall#F-measure">Precision</a></dt>
<dd>[9.18%, 10.89%, 15.49%] (11.85%)</dd><dt><a href="https://en.wikipedia.org/wiki/Precision_and_recall#F-measure">Recall</a></dt>
<dd>[14.02%, 30.65%, 26.29%] (23.65%)</dd><dt><a href="https://en.wikipedia.org/wiki/F1_score">F1 score</a></dt>
<dd>[10.94%, 15.93%, 18.60%] (15.16%)</dd><dt><a href="https://en.wikipedia.org/wiki/F1_score">F2 score</a></dt>
<dd>[12.52%, 22.29%, 22.12%] (18.98%)</dd><dt><a href="https://en.wikipedia.org/wiki/Jaccard_index">Jaccard index</a></dt>
<dd>[7.13%, 10.55%, 13.97%] (10.55%)</dd><dt><a href="https://en.wikipedia.org/wiki/Information_retrieval#Average_precision">Average precision</a></dt>
<dd>[9.52%, 23.62%, 21.79%] (18.31%)</dd><dt><a href="https://en.wikipedia.org/wiki/Information_retrieval#R-Precision">R-Precision</a></dt>
<dd>[9.36%, 18.59%, 19.95%] (15.97%)</dd><dt><a href="https://en.wikipedia.org/wiki/Discounted_cumulative_gain">Discounted cumulative gain</a></dt>
<dd>[12.63%, 28.98%, 25.46%] (22.36%)</dd><dt><a href="https://en.wikipedia.org/wiki/Discounted_cumulative_gain">DCG (alternative)</a></dt>
<dd>[12.74%, 25.84%, 24.52%] (21.03%)</dd></dl>

<h2>Table</h2>
<table>

<colgroup>
<col style="width:25%">
<col style="width:20%">
<col style="width:20%">
<col style="width:20%">
<col style="width:10%">
<col style="width:5%">
</colgroup>

<thead>
<tr>
<th>Query</th>
<th>TP</th>
<th>FP</th>
<th>FN</th>
<th>Match Type</th>
<th>Score</th>
</tr>
</thead>

<tbody>

<tr class="query"><td rowspan="16">
<h3><a href="http://cpfp.sourceforge.net">CPFP</a></h3>
<p>Central Proteomics Facilities Pipeline</p>
</td></tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows (Pipelines)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.53%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3366">Data integration and warehousing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2470">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2470" class="obsolete">Data retrieval (protein family annotation) (Protein family information retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">13.57%</td>
</tr>

<tr class="row operation" title="operation_2459">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2459" class="obsolete">Structure processing (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.63%</td>
</tr>

<tr class="row operation" title="operation_2447">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2447" class="obsolete">Sequence processing (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.63%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2052">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2052">Protein family report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.39%</td>
</tr>

<tr class="row format" title="format_1580">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1580" class="obsolete">ProDom entry format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://www.broad.mit.edu/cancer/software/genepattern/download/">GenePattern</a></h3>
<p>platform for integrative genomics and proteomics (includes PEPPeR  and other tools for proteomics)</p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.44%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.44%</td>
</tr>

<tr class="row topic" title="topic_622">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0622">Genomics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_85">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0085">Functional genomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0434">Integrated gene prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row operation" title="operation_3208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3208">Genome visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row operation" title="operation_362">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0362">Genome annotation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row operation" title="operation_3432">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3432">Clustering</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.05%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.01%</td>
</tr>

<tr class="row format" title="format_1218">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1218">unambiguous pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="row format" title="format_3162">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3162">MAGE-TAB</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3709">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3709">GCT/Res format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2058">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2058">Gene expression report format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://search.cpan.org/~alexmass/InSilicoSpectro-1.3.24/">InSilicoSpectro</a></h3>
<p>open source proteomics library (of Perl functions) </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_2276">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2276" class="obsolete">Protein function prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.60%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation (Function)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score medium">39.38%</td>
</tr>

<tr class="row operation" title="operation_1778">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1778">Protein function comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.60%</td>
</tr>

<tr class="row operation" title="operation_1777">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1777">Protein function prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.60%</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3252">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3252">OWL Functional Syntax</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.14%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://kirchnerlab.github.io/libmgf/">libmgf</a></h3>
<p>a flex/bison-based Mascot Generic Format (MGF) parser library</p>
</td></tr>

<tr class="row topic" title="topic_3341">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3341">Clone library</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row topic" title="topic_3343">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3343">Compound libraries and screening (Compound library)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.88%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_282">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0282">Genetic mapping (Genetic map generation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.64%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_336">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0336">Format validation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.43%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://sing.ei.uvigo.es/mass-up/">Mass-up</a></h3>
<p>utility with full GUI for proteomics data analysis, particularly MALDI-TOF</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row operation" title="operation_273">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0273">Protein interaction raw data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.14%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_3215">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3432">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3432">Clustering</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.bioprocess.org/massypup">MASSyPup</a></h3>
<p>a lightweight Linux live distribution prepackaged with a wide range of tools for MS and MS/MS data analysis</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">46.79%</td>
</tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.63%</td>
</tr>

<tr class="row topic" title="topic_81">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0081">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.38%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.41%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.12%</td>
</tr>

<tr class="row operation" title="operation_3646">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3646">Peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.14%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.46%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.46%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="https://code.google.com/p/meta-proteome-analyzer/">MetaProteomeAnalyzer</a></h3>
<p>pipeline to analyze metaproteomics data, supporting MGF and X!Tandem, OMSSA and Mascot </p>
</td></tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows (Pipelines)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">39.58%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">20.65%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.66%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="row operation" title="operation_228">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0228" class="obsolete">Data index analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.87%</td>
</tr>

<tr class="row operation" title="operation_3646">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3646">Peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://mspire.rubyforge.org">mspire|mspire-sequest</a></h3>
<p>MS data processing in Ruby, including mzML reader/writer/converter, ''in-silico'' digestion, isotopic pattern calculation etc. </p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation (Data rendering)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.01%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.86%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.86%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">33.36%</td>
</tr>

<tr class="row operation" title="operation_2504">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2504" class="obsolete">Structural data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.31%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3244">
<td class="tp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.67%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.21%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.00%</td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3655">pepXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://github.com/frallain/MSFileReader_Python_bindings">MSFileReader_Python_bindings</a></h3>
<p>Python bindings for the (non-free) ThermoFisher Scientific MSFileReader</p>
</td></tr>

<tr class="row topic" title="topic_3534">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3534">Protein binding sites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row topic" title="topic_3125">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3125">DNA binding sites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row topic" title="topic_3511">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3511">Nucleic acid sites, features and motifs (Primer binding sites)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3677">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3677">Differential binding analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row operation" title="operation_2575">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2575">Protein binding site prediction (Binding site prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">11.88%</td>
</tr>

<tr class="row operation" title="operation_420">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0420">Protein-nucleic acid binding prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1568">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1568" class="obsolete">BIND entry format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF (Proprietary mass-spectrometry format of Thermo Scientific's ProteomeDiscoverer software.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.96%</td>
</tr>

<tr class="row format" title="format_3712">
<td class="tp"><a href="http://edamontology.org/format_3712">Thermo RAW (Proprietary file format for mass spectrometry data from Thermo Scientific.)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://mzjava.expasy.org/">MzJava</a></h3>
<p>library for the analysis of mass spectrometry data from large scale proteomics and glycomics experiments</p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.32%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.18%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.56%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.21%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.61%</td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3712">Thermo RAW (Proprietary file format for mass spectrometry data from Thermo Scientific.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">17.68%</td>
</tr>

<tr class="row format" title="format_2056">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2056">Microarray experiment data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.05%</td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://open-ms.sourceforge.net">OpenMS</a></h3>
<p>library for the analysis, reduction and visualization of LC-MS(/MS) data</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">31.45%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.30%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.57%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="row operation" title="operation_3447">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3447">Diffraction data reduction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.62%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3634">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3634">Label-free quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3635">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3635">Labeled quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.66%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://pappso.inra.fr/bioinfo/">PAPPSO</a></h3>
<p>Plateforme d'Analyse Prot√©omique de Paris Sud-Ouest</p>
</td></tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.09%</td>
</tr>

<tr class="row topic" title="topic_3656">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3656">Immunoprecipitation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.82%</td>
</tr>

<tr class="row topic" title="topic_3516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3516">Genotyping experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.82%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row operation" title="operation_524">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0524">De-novo assembly (De Bruijn graph)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">11.88%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3312">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3312">GDE</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.72%</td>
</tr>

<tr class="row format" title="format_3167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3167">Experiment annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.04%</td>
</tr>

<tr class="row format" title="format_1928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1928">Staden experiment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.04%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2332">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2332">XML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="20">
<h3><a href="http://pcarvalho.com/patternlab/">PatternLab</a></h3>
<p>suite of pattern recognition software for interpretation of quantiative proteomics data  </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_2225">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2225" class="obsolete">Protein databases (Protein data resources)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Recognition)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.92%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3634">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3634">Label-free quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3635">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3635">Labeled quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3712">Thermo RAW</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://pfind.ict.ac.cn/index.html">pFind Studio</a></h3>
<p>computational platform for mass spectrometry-based proteomics, including pFind , pNovo  and pQuant </p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.61%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">30.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation (Computational subroutine)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">20.63%</td>
</tr>

<tr class="row operation" title="operation_3557">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3557">Imputation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.89%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.38%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3630">Quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.86%</td>
</tr>

<tr class="row format" title="format_3681">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3681">mzTab (mzTab is a tab-delimited format for mass spectrometry-based proteomics and metabolomics results.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.76%</td>
</tr>

<tr class="row format" title="format_3683">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3683">qcML (The focus of qcML is towards mass spectrometry based proteomics, but the format is suitable for metabolomics and sequencing as well.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">11.94%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://peptide-shaker.googlecode.com">PeptideShaker</a></h3>
<p>platform for interpretation of proteomics identification results from multiple search engines </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.92%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.39%</td>
</tr>

<tr class="row operation" title="operation_3703">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3703">Reference identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.06%</td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.34%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="https://code.google.com/p/pride-toolsuite/">PRIDE Toolsuite</a></h3>
<p>a selection of tools and libraries for interacting with data in PRIDE</p>
</td></tr>

<tr class="row topic" title="topic_602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0602">Molecular interactions, pathways and networks (Interactions)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3526">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3526" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_147">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0147" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_219">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0219">Data submission, annotation and curation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2993">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2993" class="obsolete">Molecular interaction data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row operation" title="operation_2405">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2405" class="obsolete">Protein interaction data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row operation" title="operation_3431">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3431">Deposition</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format (Format for molecular interaction data.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2332">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2332">XML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.proteios.org">Proteios</a></h3>
<p>pipeline/LIMS for proteomics experiments and analysis</p>
</td></tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">52.50%</td>
</tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows (Pipelines)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">42.75%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.21%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.21%</td>
</tr>

<tr class="row operation" title="operation_3223">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3223">Differential expression analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.52%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3167">Experiment annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="row format" title="format_1928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1928">Staden experiment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.86%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://github.com/specht/proteomatic">Proteomatic</a></h3>
<p>platform for creating MS/MS data analysis workflows using scripts </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">43.21%</td>
</tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.68%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.46%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.52%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.27%</td>
</tr>

<tr class="row operation" title="operation_2500">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2500" class="obsolete">Microarray raw data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.27%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3696">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3696">PS (PostScript)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.13%</td>
</tr>

<tr class="row format" title="format_2032">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2032">Workflow format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.18%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.59%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://proteowizard.sourceforge.net">ProteoWizard</a></h3>
<p>open source library for proteomics tools development (supports mzML) </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row operation" title="operation_2420">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2420" class="obsolete">Operation (typed)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.72%</td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata) (Tool information retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">10.89%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row format" title="format_3682">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3682">imzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://pymzml.github.com">pymzML</a></h3>
<p>Python module to parse mzML data based on cElementTree </p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.07%</td>
</tr>

<tr class="row topic" title="topic_3038">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3038" class="obsolete">Structure databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_225">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0225" class="obsolete">Data retrieval (database cross-reference)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.05%</td>
</tr>

<tr class="row operation" title="operation_516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0516" class="obsolete">Data retrieval (database metadata)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.05%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.45%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3244">
<td class="tp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data model)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">23.14%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.14%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://packages.python.org/pyteomics/">Pyteomics</a></h3>
<p>framework for proteomics data analysis, supporting mzML, MGF, pepXML and more </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_3244">
<td class="tp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://qupe.cebitec.uni-bielefeld.de">QuPE</a></h3>
<p>integrated environment for storage, analysis and integration of proteomics data (requires login) </p>
</td></tr>

<tr class="row topic" title="topic_3366">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3366">Data integration and warehousing (Data integration)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">31.41%</td>
</tr>

<tr class="row topic" title="topic_3572">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3572">Data quality management (Data integrity)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">31.41%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row operation" title="operation_3453">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3453">Diffraction data integration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.04%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.01%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.51%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.89%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.89%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="https://github.com/NCIP/cagrid-general/tree/master/demos/06annual/workflow/JRProteomicsGridSkeletonValue-1.0/src/org/globus/cagrid/RProteomics">Rproteomics</a></h3>
<p>set of routines for analyzing proteomics data, an XML database to store the results and a user interface</p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.22%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.60%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.02%</td>
</tr>

<tr class="row operation" title="operation_2405">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2405" class="obsolete">Protein interaction data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.58%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2332">XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row format" title="format_3331">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3331">BLAST XML results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.83%</td>
</tr>

<tr class="row format" title="format_3157">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3157">EBI Application Result XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.05%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="25">
<h3><a href="http://tools.proteomecenter.org/software.php">TPP</a></h3>
<p>Institute for Systems Biology "Trans-Proteomic Pipeline"</p>
</td></tr>

<tr class="row topic" title="topic_2259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2259">Systems biology</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">52.50%</td>
</tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows (Pipelines)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">42.75%</td>
</tr>

<tr class="row topic" title="topic_3070">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3070">Biology</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3545">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3545" class="obsolete">Mathematical modelling (Model some biological system using mathematical techniques including  dynamical systems, statistical models, differential equations, and game theoretic models.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">8.36%</td>
</tr>

<tr class="row operation" title="operation_2426">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2426">Modelling and simulation (Model or simulate some biological entity or system, typically using mathematical techniques including  dynamical systems, statistical models, differential equations, and game theoretic models.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row operation" title="operation_3258">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3258">Transcriptome assembly</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.83%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3436">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3436">Aggregation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3646">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3646">Peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3645">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3645">PTM identification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3635">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3635">Labeled quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML (Common file format for proteomics mass spectrometric data developed at the Seattle Proteome Center/Institute for Systems Biology.)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">17.68%</td>
</tr>

<tr class="row format" title="format_3155">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3155">SBRML (Systems Biology Result Markup Language (SBRML), the standard XML format for simulated or calculated results (e.g. trajectories) of systems biology models.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">11.91%</td>
</tr>

<tr class="row format" title="format_3692">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3692">SBGN-ML (SBGN-ML is an XML format for Systems Biology Graphical Notation (SBGN) diagrams of biological pathways or networks.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">11.37%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3247">mzIdentML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3468">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3468">xls</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://metlin.scripps.edu/xcms/">XCMS</a></h3>
<p>software package (in R) for metabolite profiling from LC-MS data</p>
</td></tr>

<tr class="row topic" title="topic_79">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0079" class="obsolete">Metabolites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.10%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Metabolites)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.89%</td>
</tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row topic" title="topic_3172">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3172">Metabolomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.78%</td>
</tr>

<tr class="row operation" title="operation_537">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0537" class="obsolete">Protein structure assignment (from NMR data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.49%</td>
</tr>

<tr class="row operation" title="operation_297">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0297">3D profile generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.33%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://cdfread.sourceforge.net/">cdfread</a></h3>
<p>a simple reader of mass spectra in netCDF</p>
</td></tr>

<tr class="row topic" title="topic_3277">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3277">Sample collections (samples)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">24.30%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.75%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.00%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.50%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3650">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3650">netCDF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row format" title="format_2001">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2001">EMBOSS simple format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.00%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.biomechanic.org/comspari">COMSPARI</a></h3>
<p>compares two datasets in netCDF or ASCII format</p>
</td></tr>

<tr class="row topic" title="topic_797">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0797">Comparative genomics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_3308">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3308">Transcriptomics (Comparative transcriptomics)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_2275">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2275">Molecular modelling (Comparative modelling)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0561" class="obsolete">Sequence formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.81%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.81%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://imdevsoftware.wordpress.com/">imDEV</a></h3>
<p>data exploration and visualization in Excel</p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row operation" title="operation_2422">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2422">Data retrieval</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_1812">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_1812">Loading</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="Insilicos Viewer">Insilicos Viewer</a></h3>
<p>basic viewer of LC-MS/MS data in mzXML format</p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.88%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.88%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row operation" title="operation_561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0561" class="obsolete">Sequence formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">46.77%</td>
</tr>

<tr class="row format" title="format_3706">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3706">Biodiversity data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.73%</td>
</tr>

<tr class="row format" title="format_2210">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2210" class="obsolete">Strain data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.73%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://arep.med.harvard.edu/MapQuant/index.html">MapQuant</a></h3>
<p>image analysis" of LC-MS features, deconvolution etc. </p>
</td></tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.39%</td>
</tr>

<tr class="row topic" title="topic_218">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0218">Text mining (Literature analysis)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">17.45%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3443">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3443">Image analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">58.33%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3629">Deisotoping (Deconvolution)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1978">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1978">DASGFF (das feature)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.05%</td>
</tr>

<tr class="row format" title="format_3547">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3547">Image format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1981">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1981" class="obsolete">GenBank feature</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.massxpert.org">massXpert</a></h3>
<p>tool for analysis of polymer and peptide/protein mass spectra</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.76%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">36.76%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.71%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">15.71%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.mmass.org/">mMass</a></h3>
<p>user-friendly and free software to view MS data in a range of formats </p>
</td></tr>

<tr class="row topic" title="topic_3071">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3071">Data management</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.58%</td>
</tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.08%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0561" class="obsolete">Sequence formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">45.38%</td>
</tr>

<tr class="row format" title="format_2078">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2078">Sequence range format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.10%</td>
</tr>

<tr class="row format" title="format_3706">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3706">Biodiversity data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.27%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="12">
<h3><a href="http://www.moleculardetective.org/FPLibrary/FPLibraryMenu.html">MoverZ</a></h3>
<p>free version of MoverZ for viewing and manipulating single mass spectra in many formats</p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.63%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">13.50%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.50%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td class="tp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.38%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.31%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.31%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://proteowizard.sourceforge.net/tools/msaccess.html">msaccess</a></h3>
<p>creates a psuedo-2D-gel representation </p>
</td></tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging (The visual representation of an object.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="row topic" title="topic_3371">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3371">Synthetic chemistry (The use of chemistry to create new compounds.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.67%</td>
</tr>

<tr class="row topic" title="topic_749">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0749">Transcription factors and regulatory sites (CAAT signals)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.63%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_370">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0370">Restriction site creation (Create (or remove) restriction sites in sequences, for example using silent mutations.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="row operation" title="operation_3219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3219">Read pre-processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row operation" title="operation_3198">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3198">Read mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3462">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3462">CRAM</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="row format" title="format_1370">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1370">HMMER format (Format of a hidden Markov model representation used by the HMMER package.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.42%</td>
</tr>

<tr class="row format" title="format_2334">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2334" class="obsolete">URI format (Typical textual representation of a URI.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://msgraph.sourceforge.net/">MSGraph</a></h3>
<p>an MS-DOS program to visualize LC-MS data in 2D and 3D (converter from mzXML available)</p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.12%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.90%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.56%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.91%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.08%</td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.48%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.36%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.73%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="MSPicturePerfect">MSPicturePerfect</a></h3>
<p>visualize mzXML data in 2D</p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.21%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.66%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.53%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.60%</td>
</tr>

<tr class="row operation" title="operation_2422">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2422">Data retrieval</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.66%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.48%</td>
</tr>

<tr class="row format" title="format_3706">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3706">Biodiversity data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.49%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://mzmine.sourceforge.net">MZmine</a></h3>
<p>toolbox for visualization and analysis of LC-MS data in netCDF or mzXML </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">30.33%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.85%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3650">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3650">netCDF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://www.ms-utils.org/mzXMLplot/index.html">mzXMLplot</a></h3>
<p>plots mzXML LC-MS data as virtual gels or heat maps</p>
</td></tr>

<tr class="row topic" title="topic_102">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0102">Mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_3345">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3345">Data identity and mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.65%</td>
</tr>

<tr class="row topic" title="topic_796">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0796" class="obsolete">Genetic mapping and linkage (Linkage mapping)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.27%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row operation" title="operation_2429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2429">Mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row operation" title="operation_531">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0531">Heat map generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3285">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3285">MAP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_3288">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3288">PED/MAP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row format" title="format_3603">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3603">PNG</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/mzXMLViewer.php">mzXML Viewer</a></h3>
<p>mzXML viewer from Seattle Proteome Center</p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2241">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2241">Transmembrane protein visualisation (Transmembrane protein rendering)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.55%</td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.86%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row format" title="format_3158">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3158">PSI MI XML (MIF)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.19%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://sourceforge.net/projects/openchrom">OpenChrom</a></h3>
<p>open source tool for mass spectrometry and chromatography </p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3694">
<td class="tp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.78%</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3712">Thermo RAW (Proprietary file format for mass spectrometry data from Thermo Scientific.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_3682">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3682">imzML (imzML is a data format for mass spectrometry imaging data. NB.: See comment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="19">
<h3><a href="http://tools.proteomecenter.org/Pep3D.php">Pep3D</a></h3>
<p>plots LC-MS and LC-MS/MS data in a 2D ''m''/''z'' vs time plot </p>
</td></tr>

<tr class="row topic" title="topic_3345">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3345">Data identity and mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.28%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.34%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.34%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.32%</td>
</tr>

<tr class="row operation" title="operation_490">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0490">Dotplot plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.68%</td>
</tr>

<tr class="row operation" title="operation_2943">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2943">Microarray Box-Whisker plot plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.43%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.43%</td>
</tr>

<tr class="row format" title="format_3706">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3706">Biodiversity data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.18%</td>
</tr>

<tr class="row format" title="format_2210">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2210" class="obsolete">Strain data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.18%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3603">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3603">PNG</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.geneffects.com/peppy/">Peppy</a></h3>
<p>proteogenomic, proteomic search tool </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.80%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.80%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.30%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row operation" title="operation_343">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0343" class="obsolete">Transmembrane protein database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.47%</td>
</tr>

<tr class="row operation" title="operation_340">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0340" class="obsolete">Protein secondary database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.08%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.06%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">13.48%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="https://www.proteomicsdb.org/">ProteomicsDB</a></h3>
<p>explorable, interactive human proteome database, including MS/MS data </p>
</td></tr>

<tr class="row topic" title="topic_2225">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2225" class="obsolete">Protein databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Protein databases)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2405">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2405" class="obsolete">Protein interaction data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.97%</td>
</tr>

<tr class="row operation" title="operation_516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0516" class="obsolete">Data retrieval (database metadata)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.35%</td>
</tr>

<tr class="row operation" title="operation_529">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0529" class="obsolete">MPSS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.56%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://pappso.inra.fr/bioinfo/proticdb/">PROTICdb</a></h3>
<p>proteomic database to store, track, query and compare proteome data </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.31%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.31%</td>
</tr>

<tr class="row topic" title="topic_2225">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2225" class="obsolete">Protein databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.73%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.60%</td>
</tr>

<tr class="row operation" title="operation_224">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0224">Query and retrieval (Query)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.03%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.42%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://wlab.ethz.ch/protter/">Protter</a></h3>
<p>interactive protein feature visualization and integration with experimental proteomic data </p>
</td></tr>

<tr class="row topic" title="topic_128">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0128">Protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.40%</td>
</tr>

<tr class="row topic" title="topic_602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0602">Molecular interactions, pathways and networks (Interactions)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">37.97%</td>
</tr>

<tr class="row topic" title="topic_3074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3074" class="obsolete">Protein feature detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.71%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2405">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2405" class="obsolete">Protein interaction data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.95%</td>
</tr>

<tr class="row operation" title="operation_3092">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3092">Protein feature detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.71%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">32.51%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.99%</td>
</tr>

<tr class="row format" title="format_2542">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2542" class="obsolete">Protein features (domains) format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.proteomesoftware.com/Scaffold/Scaffold_viewer.htm">Scaffold Viewer</a></h3>
<p>free viewer to share and visualize Scaffold-analyzed data</p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.17%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.05%</td>
</tr>

<tr class="row operation" title="operation_1830">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1830">Free cysteine detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://bio.informatik.uni-jena.de/sirius">Sirius</a></h3>
<p>tool for metabolite identification using mass spectrometry</p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">61.61%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">55.45%</td>
</tr>

<tr class="row topic" title="topic_79">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0079" class="obsolete">Metabolites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.15%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.89%</td>
</tr>

<tr class="row operation" title="operation_3703">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3703">Reference identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.52%</td>
</tr>

<tr class="row format" title="format_3250">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3250">spML (spML is the format for describing proteomics sample processing, other than using gels, prior to mass spectrometric protein identification, standardised by HUPO PSI PS. It may also be applicable for metabolomics.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">15.11%</td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3712">Thermo RAW (Proprietary file format for mass spectrometry data from Thermo Scientific.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.92%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://brendanx-uw1.gs.washington.edu/labkey/project/home/software/Skyline/begin.view">Skyline</a></h3>
<p>builds SRM/MRM methods and analyzes resulting data</p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.90%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.67%</td>
</tr>

<tr class="row topic" title="topic_606">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0606" class="obsolete">Literature data resources</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.01%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3636">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3636">MRM/SRM</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.18%</td>
</tr>

<tr class="row operation" title="operation_3447">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3447">Diffraction data reduction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.84%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.45%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://omics.pnl.gov/software/spectrumlook">Spectrum Look</a></h3>
<p>tool for viewing and annotating MS/MS spectra</p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_221">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0221" class="obsolete">Sequence annotation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.31%</td>
</tr>

<tr class="row topic" title="topic_222">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0222" class="obsolete">Genome annotation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.31%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_226">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0226">Annotation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.81%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.88%</td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata) (Data retrieval (tool annotation))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.22%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF (Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_3167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3167">Experiment annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.15%</td>
</tr>

<tr class="row format" title="format_2031">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2031">Gene annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.15%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.taverna.org.uk/">Taverna</a></h3>
<p>powerful, scalable, open source and domain independent tools for designing and executing workflows </p>
</td></tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row topic" title="topic_3539">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3539" class="obsolete">Protein domains</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_150">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0150" class="obsolete">Protein design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2430">Design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row operation" title="operation_332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0332" class="obsolete">Immunogen design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.50%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2032">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2032">Workflow format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row format" title="format_1665">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1665">Taverna workflow format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.83%</td>
</tr>

<tr class="row format" title="format_1563">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1563" class="obsolete">SMART domain assignment report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://ms-utils.org/Taverna/">Taverna4Proteomics</a></h3>
<p>Taverna workflows for proteomics  </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2459">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2459" class="obsolete">Structure processing (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.10%</td>
</tr>

<tr class="row operation" title="operation_2447">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2447" class="obsolete">Sequence processing (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.10%</td>
</tr>

<tr class="row operation" title="operation_3023">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3023">Prediction and recognition (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.71%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1665">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1665">Taverna workflow format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">58.33%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_2032">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2032">Workflow format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://omics.pnl.gov/software/viper">VIPER</a></h3>
<p>visualizes features from LC-MS analyses, particularly AMTs</p>
</td></tr>

<tr class="row topic" title="topic_3512">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3512">Gene transcripts (mRNA features)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="row topic" title="topic_114">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0114">Gene structure (Gene features)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="row topic" title="topic_160">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0160">Sequence sites, features and motifs (Sequence features)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.71%</td>
</tr>

<tr class="row operation" title="operation_2408">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2408" class="obsolete">Sequence feature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.30%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.41%</td>
</tr>

<tr class="row operation" title="operation_3203">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3203">Chromatogram visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1978">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1978">DASGFF (das feature)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.48%</td>
</tr>

<tr class="row format" title="format_1981">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1981" class="obsolete">GenBank feature</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row format" title="format_1977">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1977" class="obsolete">swiss feature</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="WinICR">WinICR</a></h3>
<p>reads and analyzes single FTICR and TOF spectra in various older vendor formats</p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.63%</td>
</tr>

<tr class="row topic" title="topic_184">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0184" class="obsolete">Threading</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.75%</td>
</tr>

<tr class="row topic" title="topic_3063">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3063">Medical informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row operation" title="operation_561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0561" class="obsolete">Sequence formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.31%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.31%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.wehi.edu.au/people/andrew-webb/1298/apl-mgf-converter">APLToMGFConverter</a></h3>
<p>converts MaxQuant APL (Andromeda peak lists) to MGF</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Content management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.30%</td>
</tr>

<tr class="row topic" title="topic_3277">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3277">Sample collections</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.78%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.51%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.61%</td>
</tr>

<tr class="row operation" title="operation_3215">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row operation" title="operation_3222">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3222">Peak calling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3714">
<td class="tp"><a href="http://edamontology.org/format_3714">MaxQuant APL peaklist format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.77%</td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3008">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3008">MAF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/collect_mgf.c">collect_mgf</a></h3>
<p>collects [[http://www.matrixscience.com/help/data_file_help.html|MGF files and dd_results from an XMass setup_QDD.tcl experiment to a single MGF file</p>
</td></tr>

<tr class="row topic" title="topic_3524">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3524">Simulation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="row topic" title="topic_3656">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3656">Immunoprecipitation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File reformatting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="row operation" title="operation_3359">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3359">Splitting (File splitting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.93%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.96%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="https://www.bruker.com/service/support-upgrades/software-downloads/mass-spectrometry.html">CompassXport</a></h3>
<p>converts Bruker analysis.baf and analysis.yep files to mzXML or mzML</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.84%</td>
</tr>

<tr class="row topic" title="topic_81">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0081">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.84%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.61%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.39%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3244">
<td class="tp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.43%</td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">33.21%</td>
</tr>

<tr class="row format" title="format_3682">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3682">imzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ce4csb.org/software.shtml">dat2mgf</a></h3>
<p>converts Mascot results files back to MGF</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.63%</td>
</tr>

<tr class="row topic" title="topic_204">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0204">Gene regulation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.56%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.36%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row format" title="format_3713">
<td class="tp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.53%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/DataAnalysis2TPP.html">DataAnalysis2TPP</a></h3>
<p>converts MGF from Bruker DataAnalysis to TPP-friendly format for use with XPRESS and ASAPRatio</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">30.38%</td>
</tr>

<tr class="row topic" title="topic_654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0654">DNA (DNA analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.40%</td>
</tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.86%</td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.56%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.86%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://proteowizard.sourceforge.net/tools/idconvert.html">idconvert</a></h3>
<p>converts between pepXML and mzIdentML</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Content management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">10.85%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">6.03%</td>
</tr>

<tr class="row topic" title="topic_220">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0220" class="obsolete">Document, record and content management</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.36%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.48%</td>
</tr>

<tr class="row operation" title="operation_233">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0233">Sequence conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.39%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (Format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">12.72%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_3248">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3248">mzQuantML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2332">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2332">XML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.quadtechassociates.com/freeware.html">MassHunter File Reader</a></h3>
<p>access raw data from Agilent MassHunter GC/MS and GC/MS/MS, export to plain text</p>
</td></tr>

<tr class="row topic" title="topic_218">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0218">Text mining (Text data mining)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">18.21%</td>
</tr>

<tr class="row topic" title="topic_749">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0749">Transcription factors and regulatory sites (GC signals)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">16.55%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2500">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2500" class="obsolete">Microarray raw data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row operation" title="operation_273">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0273">Protein interaction raw data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.36%</td>
</tr>

<tr class="row operation" title="operation_3289">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3289">ID retrieval (Accession retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.36%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2330">Textual format (Plain text)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="row format" title="format_1964">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1964">plain text format (unformatted)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://tools.proteomecenter.org/MassWolf.php">MassWolf</a></h3>
<p>converts MassLynx format to mzXML</p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Content management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">10.85%</td>
</tr>

<tr class="row topic" title="topic_3063">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3063">Medical informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.48%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (Format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.81%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row format" title="format_3309">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3309">CT (Connect format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">33.50%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://omics.pnl.gov/software/concatenated-dta-mascot-generic-file-mgf-file-converter">.dta to MGF File Converter</a></h3>
<p>converts .dta to MGF</p>
</td></tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.74%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Content management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">12.05%</td>
</tr>

<tr class="row topic" title="topic_654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0654">DNA</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.11%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.71%</td>
</tr>

<tr class="row operation" title="operation_233">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0233">Sequence conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.88%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (Format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.14%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row format" title="format_3652">
<td class="tp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row format" title="format_3008">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3008">MAF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.quadtechassociates.com/freeware.html">MSD File Reader</a></h3>
<p>read, display and save data from the original HP MSD based on the HP Pascal Chemstation</p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_528">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0528" class="obsolete">SAGE data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_3219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3219">Read pre-processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_3198">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3198">Read mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3214">Spectral analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">13.85%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.54%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.54%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://sourceforge.net/projects/multiplierz/">multiplierz</a></h3>
<p>scriptable framework for access to manufacturers' formats via mzAPI </p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row topic" title="topic_3393">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3393">Quality affairs (Good manufacturing practice)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">10.71%</td>
</tr>

<tr class="row topic" title="topic_102">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0102">Mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.57%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_3282">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3282">ID mapping (Accession mapping)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.85%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.88%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.88%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://proteowizard.sourceforge.net/technical/tools/msconvert.html">msconvert</a></h3>
<p>converts from RAW/WIFF/T2D/Agilent/Bruker/Waters/mzML/mzXML/MGF/MS2 to mzML/mzXML/MGF/MS2 </p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Content management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">8.14%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.12%</td>
</tr>

<tr class="row topic" title="topic_213">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0213">Mice or rats</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">4.69%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.46%</td>
</tr>

<tr class="row operation" title="operation_233">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0233">Sequence conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.04%</td>
</tr>

<tr class="row operation" title="operation_1914">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1914" class="obsolete">Structure retrieval (water)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.90%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_3244">
<td class="tp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://ms-utils.org/msgfdb2pepxml/index.html">msgfdb2pepxml</a></h3>
<p>converter from MS-GFDB output to pepXML</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Content management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">10.13%</td>
</tr>

<tr class="row topic" title="topic_3474">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3474">Machine learning (A topic concerning the application of artificial intelligence methods to algorithms, in order to create methods that can learn from data in order to generate an ouput, rather than relying on explicitly encoded information only.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.33%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.79%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="row operation" title="operation_543">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0543">Phylogenetic tree construction (from polymorphism data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row operation" title="operation_542">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0542">Phylogenetic tree generation (from gene frequencies) (Phylogenetic tree construction (from gene frequencies))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.51%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3655">
<td class="tp"><a href="http://edamontology.org/format_3655">pepXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row format" title="format_3160">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3160">NeXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row format" title="format_3309">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3309">CT (Connect format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">10.69%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ce4csb.org/software.shtml">mz2mgf</a></h3>
<p>converts mzData files to MGF</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">16.88%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.16%</td>
</tr>

<tr class="row topic" title="topic_3518">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3518">Microarray experiment (This might specify which raw data file relates to which sample and information on hybridisations, e.g. which are technical and which are biological replicates.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">7.72%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.48%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.81%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.61%</td>
</tr>

<tr class="row format" title="format_1641">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1641">affymetrix-exp (Affymetrix experimental conditions data file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.56%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://tools.proteomecenter.org/mzBruker.php">mzBruker</a></h3>
<p>converts Bruker analysis.baf files to mzXML</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.75%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.79%</td>
</tr>

<tr class="row topic" title="topic_220">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0220" class="obsolete">Document, record and content management</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">4.82%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.71%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.00%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.36%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://tools.proteomecenter.org/mzStar.php">mzStar</a></h3>
<p>converts SCIEX/ABI Analyst format (WIFF) to mzXML</p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row topic" title="topic_3370">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3370">Analytical chemistry</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.48%</td>
</tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.48%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.61%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.41%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3710">
<td class="tp"><a href="http://edamontology.org/format_3710">WIFF format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.18%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/mzxml2other.php">mzXML2Other</a></h3>
<p>converts mzXML to SEQUEST .dta, MGF, and Micromass .pkl</p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.27%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.24%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">11.63%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.61%</td>
</tr>

<tr class="row operation" title="operation_233">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0233">Sequence conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.26%</td>
</tr>

<tr class="row operation" title="operation_2503">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2503" class="obsolete">Sequence data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.58%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3653">
<td class="tp"><a href="http://edamontology.org/format_3653">pkl</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_3652">
<td class="tp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://tools.proteomecenter.org/out2summary.php">Out2Summary</a></h3>
<p>collects Sequest and TurboSequest *.out files to HTML for use with INTERACT</p>
</td></tr>

<tr class="row topic" title="topic_602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0602">Molecular interactions, pathways and networks (Interactions)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">40.91%</td>
</tr>

<tr class="row topic" title="topic_3526">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3526" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.70%</td>
</tr>

<tr class="row topic" title="topic_147">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0147" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.70%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2121">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2121">Sequence file editing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.11%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File reformatting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.58%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.49%</td>
</tr>

<tr class="row format" title="format_3556">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3556">MIME  HTML format for Web pages, which can include external resources, including images, Flash animations and so on.  (MHTML)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.03%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.uib.no/en/rg/probe/66698/pklfilemerger">PklFileMerger</a></h3>
<p>merges individual Q-TOF .pkl files into a single file for database searching</p>
</td></tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.89%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.89%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search (Sequence database search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">36.94%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">53.47%</td>
</tr>

<tr class="row operation" title="operation_339">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0339">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.89%</td>
</tr>

<tr class="row operation" title="operation_338">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0338">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.89%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3653">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3653">pkl</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.92%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (File format)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">16.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="https://code.google.com/p/pyms/">PyMS</a></h3>
<p>tools for reading LC-MS data in the ANDI-MS and JCAMP-DX formats </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row topic" title="topic_3500">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3500">Animals</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.14%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row operation" title="operation_561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0561" class="obsolete">Sequence formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.91%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.91%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://tools.proteomecenter.org/ReAdW.php">ReAdW</a></h3>
<p>converts Xcalibur native acquisition files to mzXML</p>
</td></tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.63%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.36%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td class="tp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.53%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.pepchem.org">T2D converter</a></h3>
<p>converts ABI SCIEX 4700/4800 t2d files to mzXML</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row topic" title="topic_3344">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3344">Biomedical science (Topic concerning biological science that is (typically) performed in the context of medicine.)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">5.80%</td>
</tr>

<tr class="row topic" title="topic_2640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2640">Oncology (Cancer)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.42%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.13%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (File format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.66%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row format" title="format_1628">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1628">ABI</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://code.google.com/p/unfinnigan/">unfinnigan</a></h3>
<p>reading Thermo .raw files without MsFileReader</p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_184">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0184" class="obsolete">Threading</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row topic" title="topic_3406">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3406">Ear, nose and throat medicine (Head and neck disorders)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">8.44%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3219">Read pre-processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_3198">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3198">Read mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_3230">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3230">Read depth analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3712">Thermo RAW</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.53%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://sourceforge.net/projects/protms">wiff2dta</a></h3>
<p>converts ABI WIFF to .dta </p>
</td></tr>

<tr class="row topic" title="topic_654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0654">DNA</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">13.36%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.05%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.48%</td>
</tr>

<tr class="row operation" title="operation_233">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0233">Sequence conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.39%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting (Format conversion)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">12.72%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row format" title="format_1628">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1628">ABI</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_3710">
<td class="tp"><a href="http://edamontology.org/format_3710">WIFF format (wiff)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="X2XML">NA</a></h3>
<p>converts from almost any format (Thermo, Bruker, Agilent and Micromass) to mzXML </p>
</td></tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row topic" title="topic_3500">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3500">Animals</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.75%</td>
</tr>

<tr class="row topic" title="topic_3063">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3063">Medical informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row operation" title="operation_1848">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1848" class="obsolete">Structure formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.81%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="https://wiki.mpi-cbg.de/wiki/lipidx/index.php/Main_Page">LipidXplorer</a></h3>
<p>high-throughput lipid identification from shotgun mass spectra in mzML, mzXML, CSV and .dta formats </p>
</td></tr>

<tr class="row topic" title="topic_153">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0153">Lipids</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing (High throughput sequencing)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row topic" title="topic_2842">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2842" class="obsolete">High-throughput sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row topic" title="topic_3172">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3172">Metabolomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.63%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row operation" title="operation_3357">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3357">Format detection (Format identification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.63%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.89%</td>
</tr>

<tr class="row format" title="format_2002">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2002">srs format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.68%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://lababi.bioprocess.org/index.php/lababi-software/80-spidermass">SpiderMass</a></h3>
<p>semantic chemical database generation, metabolite identification and ''de novo'' formula generation</p>
</td></tr>

<tr class="row topic" title="topic_2816">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2816" class="obsolete">Gene resources (Gene database)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.92%</td>
</tr>

<tr class="row topic" title="topic_79">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0079" class="obsolete">Metabolites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3172">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3172">Metabolomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.50%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_2035">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2035">Chemical formula format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.24%</td>
</tr>

<tr class="row format" title="format_1604">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1604" class="obsolete">DictyBase gene report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.02%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://omics.pnl.gov/software/DeconMSn.php">DeconMSn</a></h3>
<p>tool for accurate precursor ion monoisotopic mass determination for tandem mass spectra </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.00%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.88%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.53%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.71%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.75%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://omics.pnl.gov/software/DeconTools.php">DeconTools</a></h3>
<p>advanced peak picking of MS and LC-MS(/MS) data using THRASH and other algorithms</p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Algorithms)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">36.82%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.82%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.34%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0516" class="obsolete">Data retrieval (database metadata)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.60%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.86%</td>
</tr>

<tr class="row operation" title="operation_2422">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2422">Data retrieval</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.34%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.43%</td>
</tr>

<tr class="row format" title="format_3706">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3706">Biodiversity data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.18%</td>
</tr>

<tr class="row format" title="format_2210">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2210" class="obsolete">Strain data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.18%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/esimsa.html">esimsa</a></h3>
<p>simple deconvolution of electrospray ionization peak lists </p>
</td></tr>

<tr class="row topic" title="topic_3277">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3277">Sample collections (samples)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">23.62%</td>
</tr>

<tr class="row topic" title="topic_3524">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3524">Simulation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_84">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0084">Phylogeny (Phylogenetic simulation)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3629">
<td class="tp"><a href="http://edamontology.org/operation_3629">Deisotoping (Deconvolution)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_3215">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row operation" title="operation_3222">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3222">Peak calling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3714">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3714">MaxQuant APL peaklist format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_2001">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2001">EMBOSS simple format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_3612">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3612">ENCODE peak format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/esimsa2D.html">esimsa2D</a></h3>
<p>simple deconvolution and 2D peak picking of LC-MS data</p>
</td></tr>

<tr class="row topic" title="topic_3277">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3277">Sample collections (samples)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">23.14%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.90%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3629">
<td class="tp"><a href="http://edamontology.org/operation_3629">Deisotoping (Deconvolution)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">30.93%</td>
</tr>

<tr class="row operation" title="operation_3215">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.88%</td>
</tr>

<tr class="row operation" title="operation_3222">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3222">Peak calling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.88%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.bioprocess.org/esiprot">ESIprot</a></h3>
<p>charge state determination and molecular weight calculation for low resolution electrospray ionization data </p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.05%</td>
</tr>

<tr class="row topic" title="topic_2839">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2839" class="obsolete">Molecules</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.48%</td>
</tr>

<tr class="row topic" title="topic_2226">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2226" class="obsolete">Structure determination</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.77%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_398">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0398">Protein molecular weight calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.61%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row operation" title="operation_355">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0355" class="obsolete">Sequence database search (by molecular weight)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.86%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.45%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.45%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.03%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://informatics.mayo.edu/svn/trunk/mprc/raams/doc/index.html">libmprc</a></h3>
<p>library for analysis of high resolution mass spectrometry data (including THRASH)</p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.83%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.03%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.22%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.14%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.51%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.89%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.bioconductor.org/packages/release/bioc/html/xcms.html">massifquant</a></h3>
<p>Kalman filter based on binless peak picking and quantitation in XCMS</p>
</td></tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_3519">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3519">PCR experiment (PCR experiments, e.g. quantitative real-time PCR.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">5.73%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">4.69%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3695">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3695">Filtering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row operation" title="operation_3675">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3675">Variant filtering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row operation" title="operation_3215">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3598">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3598">xbm</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row format" title="format_3599">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3599">xpm</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row format" title="format_3612">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3612">ENCODE peak format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://cran.r-project.org/web/packages/nontarget/index.html">nontarget</a></h3>
<p>R function for compound, adducts and ion series detection using isotopic distributions</p>
</td></tr>

<tr class="row topic" title="topic_85">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0085">Functional genomics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.63%</td>
</tr>

<tr class="row topic" title="topic_1775">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_1775">Function analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.63%</td>
</tr>

<tr class="row topic" title="topic_3375">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3375">Pharmacokinetics and pharmacodynamics (Drug distribution)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">20.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.86%</td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation (Function)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">32.96%</td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Detection)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.88%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.25%</td>
</tr>

<tr class="row format" title="format_3252">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3252">OWL Functional Syntax</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.96%</td>
</tr>

<tr class="row format" title="format_2074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2074">Dirichlet distribution format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.83%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/Xtractor/">Xtractor</a></h3>
<p>extracts pre-defined peaks or regions into a uniform array</p>
</td></tr>

<tr class="row topic" title="topic_3518">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3518">Microarray experiment (Methylation array)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_1456">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_1456" class="obsolete">Protein membrane regions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_3125">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3125">DNA binding sites (Scaffold-attachment region)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3213">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3213" class="obsolete">Genome indexing (suffix arrays) (Suffix arrays)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row operation" title="operation_3211">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3211">Genome indexing (Suffix arrays)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row operation" title="operation_3215">
<td class="tp"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3612">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3612">ENCODE peak format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_3614">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3614">ENCODE broad peak format </a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_3613">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3613">ENCODE narrow peak format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="12">
<h3><a href="http://pcarvalho.com/patternlab/downloads/windows/yada/">Y.A.D.A.</a></h3>
<p>tool for deconvoluting MS/MS spectra </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_611">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0611">Electron microscopy (SEM)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">6.75%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.00%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3629">
<td class="tp"><a href="http://edamontology.org/operation_3629">Deisotoping (Deconvolution)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">33.80%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.25%</td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata) (Tool information retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.00%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF (Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1947">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1947">GCG MSF (MSF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">11.88%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://blog.binfalse.de/software/adjustpkl/">adjustPKL</a></h3>
<p>adjust (recalibrate) PKL peak lists</p>
</td></tr>

<tr class="row topic" title="topic_3423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3423">Complementary medicine (Holistic medicine)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">5.25%</td>
</tr>

<tr class="row topic" title="topic_3385">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3385">Light microscopy</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">3.50%</td>
</tr>

<tr class="row topic" title="topic_3448">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3448">Neutron diffraction (Elastic neutron scattering)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">2.40%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3215">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3215">Peak detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_3222">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3222">Peak calling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.67%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3653">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3653">pkl</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_3714">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3714">MaxQuant APL peaklist format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_3612">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3612">ENCODE peak format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.cs.toronto.edu/%7ejenn/CPM/">CPM</a></h3>
<p>alignment of time-series data [as in LC-MS(/MS)] using Continuous Profile Models [[http://www.cs.toronto.edu/%7ejenn/alignmentStudy/alignment.nips2004.pdf|</p>
</td></tr>

<tr class="row topic" title="topic_83">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0083" class="obsolete">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.18%</td>
</tr>

<tr class="row topic" title="topic_167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0167" class="obsolete">Structural (3D) profiles</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.99%</td>
</tr>

<tr class="row topic" title="topic_183">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0183" class="obsolete">Structure alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2928">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.18%</td>
</tr>

<tr class="row operation" title="operation_3565">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3565">RNA-seq time series data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.86%</td>
</tr>

<tr class="row operation" title="operation_299">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0299">3D profile-to-3D profile alignment (3D profile alignment)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.70%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data model)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row format" title="format_1921">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1921">Alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_1419">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1419">Sequence-MEME profile alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.24%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ce4csb.org/utils/idalign/csbserver.py">ID-Align</a></h3>
<p>aligns metabolite data from multiple files to single spreadsheet</p>
</td></tr>

<tr class="row topic" title="topic_83">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0083" class="obsolete">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_79">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0079" class="obsolete">Metabolites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Metabolites)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2928">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row operation" title="operation_2951">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2951" class="obsolete">Alignment processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_2946">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2946" class="obsolete">Alignment analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_3628">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3628">Chromatographic alignment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1921">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1921">Alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.49%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.97%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/msalign/index.html">msalign</a></h3>
<p>aligns LC-MS and LC-MS/MS datasets using peptides identified by MS/MS and accurate mass MS </p>
</td></tr>

<tr class="row topic" title="topic_83">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0083" class="obsolete">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">35.71%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.74%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2928">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.38%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.68%</td>
</tr>

<tr class="row operation" title="operation_3643">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3643">Tag-based peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.74%</td>
</tr>

<tr class="row operation" title="operation_3628">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3628">Chromatographic alignment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1921">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1921">Alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.63%</td>
</tr>

<tr class="row format" title="format_1993">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1993" class="obsolete">msf alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.92%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.71%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/msalign2/index.html">msalign2</a></h3>
<p>aligns 2 CE-MS or LC-MS datasets using accurate mass information </p>
</td></tr>

<tr class="row topic" title="topic_83">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0083" class="obsolete">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.84%</td>
</tr>

<tr class="row topic" title="topic_605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0605">Informatics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row topic" title="topic_183">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0183" class="obsolete">Structure alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.45%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2928">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.84%</td>
</tr>

<tr class="row operation" title="operation_2951">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2951" class="obsolete">Alignment processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.09%</td>
</tr>

<tr class="row operation" title="operation_2946">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2946" class="obsolete">Alignment analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.09%</td>
</tr>

<tr class="row operation" title="operation_3628">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3628">Chromatographic alignment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1921">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1921">Alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.09%</td>
</tr>

<tr class="row format" title="format_2554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2554">Alignment format (text)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.84%</td>
</tr>

<tr class="row format" title="format_2555">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2555">Alignment format (XML)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.84%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://obi-warp.sourceforge.net">OBI-Warp</a></h3>
<p>aligns multiple LC-MS(/MS) datasets in retention time by dynamic time warping </p>
</td></tr>

<tr class="row topic" title="topic_83">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0083" class="obsolete">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row topic" title="topic_183">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0183" class="obsolete">Structure alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_176">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0176">Molecular dynamics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2928">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.00%</td>
</tr>

<tr class="row operation" title="operation_512">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0512" class="obsolete">Sequence alignment generation (multiple profile) (Sequence profile alignment (multiple))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.94%</td>
</tr>

<tr class="row operation" title="operation_3628">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3628">Chromatographic alignment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1921">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1921">Alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row format" title="format_2554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2554">Alignment format (text)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.25%</td>
</tr>

<tr class="row format" title="format_2555">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2555">Alignment format (XML)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/recal2.html">recal2</a></h3>
<p>recalibrate LC-FTICRMS(/MS) datasets using peptides identified by MS/MS </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">38.45%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.79%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.36%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.73%</td>
</tr>

<tr class="row operation" title="operation_3643">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3643">Tag-based peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.79%</td>
</tr>

<tr class="row operation" title="operation_2929">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.33%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.20%</td>
</tr>

<tr class="row format" title="format_1947">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1947">GCG MSF (MSF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.39%</td>
</tr>

<tr class="row format" title="format_2562">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2562" class="obsolete">Amino acid identifier format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.93%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/SpecArray.php">SpecArray</a></h3>
<p>aligns multiple LC-MS datasets and much more </p>
</td></tr>

<tr class="row topic" title="topic_83">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0083" class="obsolete">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_183">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0183" class="obsolete">Structure alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_182">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0182" class="obsolete">Sequence alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2928">Alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_512">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0512" class="obsolete">Sequence alignment generation (multiple profile) (Sequence profile alignment (multiple))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row operation" title="operation_299">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0299">3D profile-to-3D profile alignment (3D profile alignment (multiple))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row operation" title="operation_3628">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3628">Chromatographic alignment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1921">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1921">Alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_2554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2554">Alignment format (text)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_2555">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2555">Alignment format (XML)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://theorchromo.ru/">BioLCCC</a></h3>
<p>an analytic model of peptide retention </p>
</td></tr>

<tr class="row topic" title="topic_3370">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3370">Analytical chemistry</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_2275">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2275">Molecular modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_418">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0418">Protein signal peptide detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="row operation" title="operation_2499">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2499">Splicing model analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.04%</td>
</tr>

<tr class="row operation" title="operation_412">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0412">Protein signal peptide detection (bacteria)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data model)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">16.88%</td>
</tr>

<tr class="row format" title="format_2034">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2034" class="obsolete">Biological model format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row format" title="format_1666">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1666" class="obsolete">BioModel mathematical model format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.hplcsimulator.org/">hplcsimulator</a></h3>
<p>a general tool for simulating and optimizing HPLC gradients</p>
</td></tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3303">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3303">Medicine (General medicine)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_3524">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3524">Simulation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_3370">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3370">Analytical chemistry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_3566">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3566">Simulated gene expression data generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.83%</td>
</tr>

<tr class="row operation" title="operation_3562">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3562">Network simulation (Network topology simulation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.36%</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_2031">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2031">Gene annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_2175">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2175" class="obsolete">Gene cluster format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://retentionprediction.org/">retentionprediction</a></h3>
<p>a related web utility for projecting/predicting retention times </p>
</td></tr>

<tr class="row topic" title="topic_2278">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2278" class="obsolete">Transmembrane protein prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.41%</td>
</tr>

<tr class="row topic" title="topic_82">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0082">Structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.60%</td>
</tr>

<tr class="row topic" title="topic_109">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0109" class="obsolete">Gene finding (Gene prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.72%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3633">
<td class="tp"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.64%</td>
</tr>

<tr class="row operation" title="operation_440">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0440">Promoter prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.55%</td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Prediction)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.09%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3556">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3556">MIME  HTML format for Web pages, which can include external resources, including images, Flash animations and so on. </a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.86%</td>
</tr>

<tr class="row format" title="format_1297">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1297">EMBOSS repeat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.55%</td>
</tr>

<tr class="row format" title="format_1702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1702" class="obsolete">ChemSpider entry format (The format of an entry from a database of chemical structures and property predictions.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.28%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://www.ms-utils.org/rt.html">rt</a></h3>
<p>(legacy) retention time prediction using LC-MS data </p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.90%</td>
</tr>

<tr class="row topic" title="topic_82">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0082">Structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="row topic" title="topic_109">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0109" class="obsolete">Gene finding (Gene prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.66%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3633">
<td class="tp"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">63.10%</td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Prediction)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.93%</td>
</tr>

<tr class="row operation" title="operation_3482">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3482">Antimicrobial resistance prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.88%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.11%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="19">
<h3><a href="http://www.ms-utils.org/rt4.html">rt4</a></h3>
<p>latest version of rt (4.0) with improved user interface</p>
</td></tr>

<tr class="row topic" title="topic_602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0602">Molecular interactions, pathways and networks (Interactions)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">19.29%</td>
</tr>

<tr class="row topic" title="topic_3526">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3526" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.05%</td>
</tr>

<tr class="row topic" title="topic_147">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0147" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.05%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation (Statistical testing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.05%</td>
</tr>

<tr class="row operation" title="operation_2871">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2871">Sequence tagged site (STS) mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.89%</td>
</tr>

<tr class="row operation" title="operation_2464">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2464">Protein-protein interaction prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.95%</td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3329">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3329">HMMER3 (HMMER profile HMM file for HMMER versions 3.x)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.14%</td>
</tr>

<tr class="row format" title="format_3328">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3328">HMMER2 (HMMER profile HMM file for HMMER versions 2.x)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.14%</td>
</tr>

<tr class="row format" title="format_1975">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1975">GFF3 (Generic Feature Format version 3 (GFF3) of sequence features.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.04%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3247">mzIdentML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3655">pepXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2332">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2332">XML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://hs2.proteome.ca/SSRCalc/SSRCalc.html">SSRCalc</a></h3>
<p>sequence-specific retention time calculator </p>
</td></tr>

<tr class="row topic" title="topic_2953">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2953" class="obsolete">Nucleic acid design (Topic for the design of nucleic acid sequences with specific conformations.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.59%</td>
</tr>

<tr class="row topic" title="topic_2839">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2839" class="obsolete">Molecules</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.70%</td>
</tr>

<tr class="row topic" title="topic_184">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0184" class="obsolete">Threading (Sequence-structure alignment)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">6.68%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3633">
<td class="tp"><a href="http://edamontology.org/operation_3633">Retention times prediction (Retention times calculation)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score good">76.20%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2014">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2014">Sequence-profile alignment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row format" title="format_1419">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1419">Sequence-MEME profile alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row format" title="format_1297">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1297">EMBOSS repeat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://crosstalkdb.bmb.sdu.dk/">CrossTalkDB</a></h3>
<p>histone modification database designed to store, analyze and compare histone modifications determined by MS </p>
</td></tr>

<tr class="row topic" title="topic_3295">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3295">Epigenetics (Histone modification)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">52.67%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.91%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data design)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.79%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3561">Database comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.36%</td>
</tr>

<tr class="row operation" title="operation_2430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2430">Design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row operation" title="operation_341">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0341" class="obsolete">Motif database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.82%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.64%</td>
</tr>

<tr class="row format" title="format_1947">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1947">GCG MSF (MSF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">12.95%</td>
</tr>

<tr class="row format" title="format_1999">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1999">scores format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.86%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.abrf.org/index.cfm/dm.home">Delta Mass</a></h3>
<p>the ABRF database of protein post-translational modifications</p>
</td></tr>

<tr class="row topic" title="topic_3541">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3541" class="obsolete">Protein post-translational modifications</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score good">73.33%</td>
</tr>

<tr class="row topic" title="topic_601">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0601">Protein modifications (Protein post-translational modification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score good">69.67%</td>
</tr>

<tr class="row topic" title="topic_2225">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2225" class="obsolete">Protein databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_417">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0417">Protein post-translation modification site prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row operation" title="operation_343">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0343" class="obsolete">Transmembrane protein database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_477">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0477">Protein modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.50%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_1948">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1948">nbrf/pir (nbrf)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.38%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://denovogui.googlecode.com">DeNovoGUI</a></h3>
<p>A nice GUI for running the PepNovo and DirecTag under Windows, Mac or Linux </p>
</td></tr>

<tr class="row topic" title="topic_787">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0787" class="obsolete">Rice</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row topic" title="topic_213">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0213">Mice or rats</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="row topic" title="topic_102">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0102">Mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.26%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.87%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.14%</td>
</tr>

<tr class="row operation" title="operation_476">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0476">Ab initio structure prediction (de novo structure prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">12.49%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3008">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3008">MAF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.26%</td>
</tr>

<tr class="row format" title="format_3001">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3001">ACE</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.26%</td>
</tr>

<tr class="row format" title="format_3285">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3285">MAP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.26%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.expasy.ch/tools/glycomod">GlycoMod</a></h3>
<p>predicts oligosaccharide structures from experimental masses</p>
</td></tr>

<tr class="row topic" title="topic_82">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0082">Structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.90%</td>
</tr>

<tr class="row topic" title="topic_172">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0172" class="obsolete">Protein structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.04%</td>
</tr>

<tr class="row topic" title="topic_178">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0178" class="obsolete">Protein secondary structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.61%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Prediction)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">39.38%</td>
</tr>

<tr class="row operation" title="operation_271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0271">Structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.90%</td>
</tr>

<tr class="row operation" title="operation_3090">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3090" class="obsolete">Protein feature prediction (from structure)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.71%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2033">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2033">Tertiary structure format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1702" class="obsolete">ChemSpider entry format (The format of an entry from a database of chemical structures and property predictions.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">10.94%</td>
</tr>

<tr class="row format" title="format_2077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2077">Protein secondary structure format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://proteomics.ucsd.edu/ProteoSAFe/">InSpecT</a></h3>
<p>blind spectral search of tandem mass spectra of peptides </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">38.57%</td>
</tr>

<tr class="row topic" title="topic_3135">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3135" class="obsolete">Signal or transit peptide</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.86%</td>
</tr>

<tr class="row topic" title="topic_3544">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3544" class="obsolete">Protein signal peptides</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.86%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.39%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.43%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">13.56%</td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format (Format for mass pectra and derived data, include peptide sequences etc.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.23%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.hairyfatguy.com/lutefisk">lutefisk</a></h3>
<p>'de novo'' peptide sequencing from MS/MS data </p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.18%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">55.73%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo) (de novo transcriptome assembly)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.17%</td>
</tr>

<tr class="row operation" title="operation_476">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0476">Ab initio structure prediction (de novo structure prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.17%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.09%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://msms.usc.edu/supplementary/msnovo">MSNovo</a></h3>
<p>'de novo'' peptide sequencing from MS/MS data </p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.18%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">55.73%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo) (de novo transcriptome assembly)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.17%</td>
</tr>

<tr class="row operation" title="operation_476">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0476">Ab initio structure prediction (de novo structure prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.17%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.09%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://cloud.rapidnovor.org/">Novor</a></h3>
<p>fast, real-time peptide ''de novo'' sequencing engine </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.75%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">59.52%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row operation" title="operation_476">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0476">Ab initio structure prediction (de novo structure prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.99%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1930">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1930">FASTQ</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row format" title="format_3605">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3605">rast</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.bioinfor.com/peaks/features/denovo.html">PEAKS De Novo</a></h3>
<p>integrated ''de novo'' peptide sequencing , PTM finder , and homology search  (demo available)</p>
</td></tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.55%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.91%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.59%</td>
</tr>

<tr class="row operation" title="operation_3218">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3218">Sequencing quality control (Sequencing QC)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.91%</td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.91%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format (das sequence format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.99%</td>
</tr>

<tr class="row format" title="format_2096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2096">unambiguous sequence</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.77%</td>
</tr>

<tr class="row format" title="format_3701">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3701">sequin</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.69%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://proteomics.ucsd.edu/ProteoSAFe/">PepNovo</a></h3>
<p>'de novo'' sequencing using probabilistic network modeling </p>
</td></tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">30.54%</td>
</tr>

<tr class="row topic" title="topic_602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0602">Molecular interactions, pathways and networks (Networks)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score good">69.05%</td>
</tr>

<tr class="row operation" title="operation_3660">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3660">Metabolic network modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row operation" title="operation_3481">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3481">Probabilistic sequence generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.59%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format (das sequence format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.32%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data model)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">21.70%</td>
</tr>

<tr class="row format" title="format_2096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2096">unambiguous sequence</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="https://github.com/chernan/popitam">Popitam</a></h3>
<p>searches for unknown post-translational modifications </p>
</td></tr>

<tr class="row topic" title="topic_601">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0601">Protein modifications (Post-translation modifications)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">59.06%</td>
</tr>

<tr class="row topic" title="topic_3541">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3541" class="obsolete">Protein post-translational modifications</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.04%</td>
</tr>

<tr class="row topic" title="topic_158">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0158" class="obsolete">Sequence motifs (Motifs)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.53%</td>
</tr>

<tr class="row operation" title="operation_417">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0417">Protein post-translation modification site prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.75%</td>
</tr>

<tr class="row operation" title="operation_239">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0239">Sequence motif recognition (Motif search)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.00%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1477">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1477">mmCIF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="row format" title="format_3158">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3158">PSI MI XML (MIF) (MIF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">8.91%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.uib.no/en/rg/probe/65847/postman">POSTMan</a></h3>
<p>post-translational modification software </p>
</td></tr>

<tr class="row topic" title="topic_601">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0601">Protein modifications (Post-translation modifications)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">65.63%</td>
</tr>

<tr class="row topic" title="topic_3541">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3541" class="obsolete">Protein post-translational modifications</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_417">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0417">Protein post-translation modification site prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.00%</td>
</tr>

<tr class="row operation" title="operation_3645">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3645">PTM identification (Identification of post-translational modifications (PTMs) of peptides/proteins in mass spectrum.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">17.19%</td>
</tr>

<tr class="row operation" title="operation_238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0238">Sequence motif discovery (Motif discovery)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">11.88%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2068">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2068">Sequence motif format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF (Proprietary mass-spectrometry format of Thermo Scientific's ProteomeDiscoverer software.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">8.93%</td>
</tr>

<tr class="row format" title="format_1477">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1477">mmCIF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.67%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="https://prosightptm.northwestern.edu/">ProSight PTM</a></h3>
<p>web portal for PTM search in top-down experiments </p>
</td></tr>

<tr class="row topic" title="topic_3656">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3656">Immunoprecipitation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_3516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3516">Genotyping experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_3518">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3518">Microarray experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_3645">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3645">PTM identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3167">Experiment annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1928">Staden experiment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_2056">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2056">Microarray experiment data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://prosightptm.northwestern.edu/ionpredictor">ProSight PTM Ion Predictor</a></h3>
<p>generate exact masses for CID/ECD fragments with PTMs</p>
</td></tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row topic" title="topic_3303">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3303">Medicine (General medicine)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.70%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_282">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0282">Genetic mapping (Genetic map generation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.43%</td>
</tr>

<tr class="row operation" title="operation_3349">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3349">Bibliography generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_3602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3602">pgm</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row format" title="format_3601">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3601">pbm</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.proteomefactory.com/1sub/s505-Sequit.htm">Sequit!</a></h3>
<p>free (non-batch) version of a ''de novo''/BLAST tool </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing (The determination of complete (typically nucleotide) sequences, including those of genomes (full genome sequencing, de novo sequencing and resequencing), amplicons and transcriptomes.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.56%</td>
</tr>

<tr class="row topic" title="topic_3577">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3577">Personalized medicine</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.89%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3644">
<td class="tp"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.71%</td>
</tr>

<tr class="row operation" title="operation_3259">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3259" class="obsolete">Transcriptome assembly (de novo)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row operation" title="operation_476">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0476">Ab initio structure prediction (de novo structure prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.99%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row format" title="format_3331">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3331">BLAST XML results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.82%</td>
</tr>

<tr class="row format" title="format_3312">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3312">GDE</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.52%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="NA">SPS</a></h3>
<p>shotgun protein sequencing assembling multiple MS/MS spectra </p>
</td></tr>

<tr class="row topic" title="topic_196">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0196">Sequence assembly</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row topic" title="topic_3043">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3043" class="obsolete">Protein sequences</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row topic" title="topic_741">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0741" class="obsolete">Protein sequence alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_310">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0310">Sequence assembly</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row operation" title="operation_3180">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3180">Sequence assembly validation (Sequence assembly QC)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.57%</td>
</tr>

<tr class="row operation" title="operation_3184">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3184">Sequence assembly visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2055">Sequence assembly format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row format" title="format_2561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2561">Sequence assembly format (text)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="https://sourceforge.net/project/showfiles.php?group_id=109078&package_id=121161">theospec</a></h3>
<p>command line tool for predicting MS/MS spectra </p>
</td></tr>

<tr class="row topic" title="topic_82">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0082">Structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="row topic" title="topic_109">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0109" class="obsolete">Gene finding (Gene prediction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.66%</td>
</tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Prediction)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.93%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.18%</td>
</tr>

<tr class="row operation" title="operation_3482">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3482">Antimicrobial resistance prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.88%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF (Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">15.18%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.31%</td>
</tr>

<tr class="row format" title="format_1947">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1947">GCG MSF (MSF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">10.74%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.unimod.org">UniMod</a></h3>
<p>database of simple protein modifications</p>
</td></tr>

<tr class="row topic" title="topic_601">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0601">Protein modifications</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">65.63%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.53%</td>
</tr>

<tr class="row topic" title="topic_166">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0166">Protein structural motifs and surfaces (Protein 3D motifs)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">42.26%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_477">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0477">Protein modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.62%</td>
</tr>

<tr class="row operation" title="operation_3561">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3561">Database comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_3644">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3644">de Novo sequencing</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_2066">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2066">Database hits (sequence) format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="12">
<h3><a href="http://www.genebio.com/products/phenyx/aldente/">Aldente</a></h3>
<p>web based peptide mass fingerprinting</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.00%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.75%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2929">
<td class="tp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score good">69.67%</td>
</tr>

<tr class="row operation" title="operation_355">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0355" class="obsolete">Sequence database search (by molecular weight) (Search sequence(s) or a sequence database for sequences which match a set of peptide masses, for example a peptide mass fingerprint from mass spectrometry.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">19.12%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_3683">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3683">qcML (The focus of qcML is towards mass spectrometry based proteomics, but the format is suitable for metabolomics and sequencing as well.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">7.08%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.matrixscience.com/cgi/search_form.pl?FORMVER=2&SEARCH=PMF">MASCOT</a></h3>
<p>free version of the Matrix Science MASCOT Peptide Mass Fingerprinting</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row topic" title="topic_3390">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3390">Nutritional science</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_3344">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3344">Biomedical science</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2929">
<td class="tp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">59.87%</td>
</tr>

<tr class="row operation" title="operation_1830">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1830">Free cysteine detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.19%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3033">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3033">Matrix format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row format" title="format_1423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1423">Phylip distance matrix</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://prospector.ucsf.edu/prospector/cgi-bin/msform.cgi?form=msfitstandard">MS-Fit</a></h3>
<p>peptide mass fingerpriting tool in the ProteinProspector suite</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">39.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_613">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0613" class="obsolete">Peptides and amino acids</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2929">
<td class="tp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">62.07%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_252">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0252">Peptide immunogenicity prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3214">Spectral analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.09%</td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="NA">PepMAPPER</a></h3>
<p>multi-experiment peptide mass fingerprinting (not to be confuser with the PepMapper tool for mapping epitopes from affinity-selected peptides)</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">38.44%</td>
</tr>

<tr class="row topic" title="topic_102">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0102">Mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="row topic" title="topic_3135">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3135" class="obsolete">Signal or transit peptide</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.71%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2929">
<td class="tp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">51.46%</td>
</tr>

<tr class="row operation" title="operation_416">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0416">Epitope mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.46%</td>
</tr>

<tr class="row operation" title="operation_2429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2429">Mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3285">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3285">MAP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="row format" title="format_3288">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3288">PED/MAP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.30%</td>
</tr>

<tr class="row format" title="format_2060">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2060">Map format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="12">
<h3><a href="http://prowl.rockefeller.edu/prowl-cgi/profound.exe">ProFound</a></h3>
<p>free web version of ProFound (peptide mass fingerprinting)</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2929">
<td class="tp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">62.20%</td>
</tr>

<tr class="row operation" title="operation_1830">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1830">Free cysteine detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.86%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.82%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.04%</td>
</tr>

<tr class="row format" title="format_2049">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2049">Phylogenetic tree report (tree distances) format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.95%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.proteinguru.com">ProteinGuru</a></h3>
<p>several on-line tools, including OLMAT for peptide mass fingerprinting</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2929">
<td class="tp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">59.87%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row operation" title="operation_355">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0355" class="obsolete">Sequence database search (by molecular weight) (Search sequence(s) or a sequence database for sequences which match a set of peptide masses, for example a peptide mass fingerprint from mass spectrometry.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.43%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.67%</td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1972">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1972">NCBI format (There are several variants of this.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">11.72%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://prospector.ucsf.edu">ProteinProspector</a></h3>
<p>the ProteinProspector suite of protein identification tools</p>
</td></tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0245">Protein structural motif recognition (Protein structural feature identification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.97%</td>
</tr>

<tr class="row operation" title="operation_3703">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3703">Reference identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_2929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://easyprot.unige.ch/">EasyProt</a></h3>
<p>graphical and software platform for the analysis of MS/MS data</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">36.90%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.57%</td>
</tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.15%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="row operation" title="operation_228">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0228" class="obsolete">Data index analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.42%</td>
</tr>

<tr class="row operation" title="operation_2929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.66%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://greylag.org/">greylag</a></h3>
<p>free MS/MS search engine</p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.22%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.06%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">32.99%</td>
</tr>

<tr class="row operation" title="operation_1830">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1830">Free cysteine detection</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.93%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.22%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML (Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">22.13%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.97%</td>
</tr>

<tr class="row format" title="format_1436">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1436">TreeBASE format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.matrixscience.com/cgi/search_form.pl?FORMVER=2&SEARCH=MIS">MASCOT</a></h3>
<p>free, web-based version of the MASCOT MS/MS Ions Search</p>
</td></tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.19%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.19%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.08%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_239">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0239">Sequence motif recognition (Motif search)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.41%</td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF (Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">11.64%</td>
</tr>

<tr class="row format" title="format_3464">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3464">JSON</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.94%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.massmatrix.net">MassMatrix</a></h3>
<p>web based tools for peptide identification and quantitation</p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.50%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">41.56%</td>
</tr>

<tr class="row operation" title="operation_3643">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3643">Tag-based peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.50%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1430">Phylip continuous quantitative characters</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_2037">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2037">Phylogenetic continuous quantitative character format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.17%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.massai.dk">MassAI</a></h3>
<p>free (Windows/WINE) tool for analysis of MS/MS, bottom-up proteomics data, including non-standard PTMs and more</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.32%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.99%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.02%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data model)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">16.71%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.39%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.39%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://sourceforge.net/projects/masswiz/">MassWiz</a></h3>
<p>peptide identification and FDR calculation </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">59.06%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">65.63%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row operation" title="operation_3643">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3643">Tag-based peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">9.65%</td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">9.48%</td>
</tr>

<tr class="row format" title="format_2562">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2562" class="obsolete">Amino acid identifier format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://proteomics.ucsd.edu/ProteoSAFe/">MS-GFDB</a></h3>
<p>peptide identification using a combination of ''de novo'' and database search </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">52.03%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.10%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.10%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">57.81%</td>
</tr>

<tr class="row operation" title="operation_3646">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3646">Peptide database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">57.29%</td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">54.69%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3686">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3686">COMBINE OMEX</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.53%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format (Format of results of a sequence database search using FASTA.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">13.84%</td>
</tr>

<tr class="row format" title="format_1334">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1334">mspcrunch (Format of results of a sequence database search using some variant of MSPCrunch.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.15%</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://omics.pnl.gov/software/ms-gf">MS-GF+</a></h3>
<p>latest version of MS-GFDB above </p>
</td></tr>

<tr class="row topic" title="topic_3577">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3577">Personalized medicine</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.04%</td>
</tr>

<tr class="row topic" title="topic_622">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0622">Genomics (Personal genomics)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">7.23%</td>
</tr>

<tr class="row topic" title="topic_2269">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2269">Statistics and probability (Statistics)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">6.03%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation (Statistical testing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.90%</td>
</tr>

<tr class="row operation" title="operation_2871">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2871">Sequence tagged site (STS) mapping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.04%</td>
</tr>

<tr class="row operation" title="operation_249">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0249">Protein geometry calculation (Torsion angle calculation)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">5.63%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3329">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3329">HMMER3 (HMMER profile HMM file for HMMER versions 3.x)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.70%</td>
</tr>

<tr class="row format" title="format_3328">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3328">HMMER2 (HMMER profile HMM file for HMMER versions 2.x)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.70%</td>
</tr>

<tr class="row format" title="format_1975">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1975">GFF3 (Generic Feature Format version 3 (GFF3) of sequence features.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.60%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://omics.pnl.gov/software/MSPolygraph.php">MSPolygraph</a></h3>
<p>open source hybrid database and spectral library MS/MS search engine  </p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.13%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.03%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.03%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.36%</td>
</tr>

<tr class="row operation" title="operation_3093">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3093" class="obsolete">Database search (by sequence)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.03%</td>
</tr>

<tr class="row operation" title="operation_339">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0339">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.03%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML (Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.29%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.11%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="https://medschool.vanderbilt.edu/msrc-bioinformatics/myrimatch-source">MyriMatch</a></h3>
<p>another free MS/MS search engine</p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.60%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.44%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.45%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.60%</td>
</tr>

<tr class="row operation" title="operation_239">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0239">Sequence motif recognition (Motif search)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">18.54%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML (Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">19.21%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.35%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.21%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://pubchem.ncbi.nlm.nih.gov/omssa">OMSSA</a></h3>
<p>mass spectrometry search algorithm from NIH (also searches ETD MS/MS data) </p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">52.08%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Algorithms)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">24.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_529">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0529" class="obsolete">MPSS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.62%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.48%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.88%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.31%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.31%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.bioinfor.com/peaks/features/peaksdb.html">PEAKS DB</a></h3>
<p>'de novo''-assisted database search </p>
</td></tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.04%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.04%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search (Sequence database search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">48.49%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">65.63%</td>
</tr>

<tr class="row operation" title="operation_339">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0339">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.04%</td>
</tr>

<tr class="row operation" title="operation_338">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0338">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.04%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file ("Raw" result file from Mascot database search.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">17.19%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format (Format of results of a sequence database search using FASTA.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.42%</td>
</tr>

<tr class="row format" title="format_3312">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3312">GDE</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://noble.gs.washington.edu/proj/percolator/">Percolator</a></h3>
<p>semi-supervised learning for peptide identification from MS/MS data </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row operation" title="operation_3643">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3643">Tag-based peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.40%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.09%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="NA">PepProbe</a></h3>
<p>web-based MS/MS search engine </p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.22%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.06%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">32.99%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.22%</td>
</tr>

<tr class="row operation" title="operation_239">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0239">Sequence motif recognition (Motif search)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML (Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">22.13%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.97%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.17%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="https://github.com/buotex/BICEPS/tree/master/libs/pepsplice">PepSplice</a></h3>
<p>cache-optimized search algorithm </p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Algorithms)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.67%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row operation" title="operation_239">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0239">Sequence motif recognition (Motif search)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results (Format of results of a sequence database search using some variant of BLAST.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.87%</td>
</tr>

<tr class="row format" title="format_1334">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1334">mspcrunch (Format of results of a sequence database search using some variant of MSPCrunch.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.87%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://prospector.ucsf.edu">ProteinProspector</a></h3>
<p>the ProteinProspector suite of protein identification tools</p>
</td></tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0245">Protein structural motif recognition (Protein structural feature identification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.97%</td>
</tr>

<tr class="row operation" title="operation_3703">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3703">Reference identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="NA">Proteome ARTworks</a></h3>
<p>predicts detectability and tandem mass spectra of peptides and addresses the protein inference problem </p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.13%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">24.75%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.71%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Prediction)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">37.13%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.50%</td>
</tr>

<tr class="row operation" title="operation_3023">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3023">Prediction and recognition (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.92%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="https://code.google.com/p/searchgui/">SearchGUI</a></h3>
<p>graphical user interface to several common search engines, such as X!Tandem, Mascot and OMSSA. </p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation (Computer graphics)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">13.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search (Search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.13%</td>
</tr>

<tr class="row operation" title="operation_305">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0305">Literature search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row operation" title="operation_239">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0239">Sequence motif recognition (Motif search)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">13.50%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.83%</td>
</tr>

<tr class="row format" title="format_3467">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3467">GIF (Graphics Interchange Format.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">10.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://proteomics.informatics.iupui.edu/software/toppic/">TopPIC</a></h3>
<p>proteoform identification/characterizaion at the proteome level using database search </p>
</td></tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.23%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.23%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search (Sequence database search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">41.07%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">57.81%</td>
</tr>

<tr class="row operation" title="operation_339">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0339">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.23%</td>
</tr>

<tr class="row operation" title="operation_338">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0338">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.23%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format (Format of results of a sequence database search using FASTA.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.44%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file ("Raw" result file from Mascot database search.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">13.28%</td>
</tr>

<tr class="row format" title="format_1334">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1334">mspcrunch (Format of results of a sequence database search using some variant of MSPCrunch.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.73%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://portugene.com/vems.html">VEMS</a></h3>
<p>platform for protein identification and characterization </p>
</td></tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.29%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">29.06%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0245">Protein structural motif recognition (Protein structural feature identification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.49%</td>
</tr>

<tr class="row operation" title="operation_3703">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3703">Reference identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.thegpm.org/TANDEM/">X!Tandem</a></h3>
<p>matches tandem mass spectra with peptide sequences</p>
</td></tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">41.56%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.50%</td>
</tr>

<tr class="row operation" title="operation_3218">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3218">Sequencing quality control (Sequencing QC)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.27%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1990">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1990">match</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row format" title="format_2096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2096">unambiguous sequence</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format (das sequence format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.79%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://promex.mpimp-golm.mpg.de/home.shtml">ProMEX</a></h3>
<p>matches tandem mass spectra against ''A. thaliana'' and ''M. truncatula'' libraries </p>
</td></tr>

<tr class="row topic" title="topic_3315">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3315">Mathematics (Maths)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_3341">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3341">Clone library</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.92%</td>
</tr>

<tr class="row topic" title="topic_3343">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3343">Compound libraries and screening (Compound library)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">20.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row operation" title="operation_3725">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3725">http://edamontology.org/operation_3725<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1990">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1990">match</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.91%</td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.03%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.peptideatlas.org/spectrast/">SpectraST</a></h3>
<p>matches peptide MS/MS spectra with library spectra</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.75%</td>
</tr>

<tr class="row topic" title="topic_3315">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3315">Mathematics (Maths)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.43%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.75%</td>
</tr>

<tr class="row operation" title="operation_2929">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.66%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row operation" title="operation_3725">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3725">http://edamontology.org/operation_3725<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1990">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1990">match</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF (Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.96%</td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.thegpm.org/hunter/index.html">X! Hunter</a></h3>
<p>matches tandem mass spectra with library spectra from same organism</p>
</td></tr>

<tr class="row topic" title="topic_621">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0621">Model organisms (Organisms)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">39.58%</td>
</tr>

<tr class="row topic" title="topic_3315">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3315">Mathematics (Maths)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_616">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0616" class="obsolete">Organelles</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.86%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.11%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.39%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.07%</td>
</tr>

<tr class="row operation" title="operation_3725">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3725">http://edamontology.org/operation_3725<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1990">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1990">match</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row format" title="format_2573">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2573">SAM</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.89%</td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.91%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/goa_gubbar/index.html">goa_gubbar</a></h3>
<p>sorts protein identifications under GOA slim terms</p>
</td></tr>

<tr class="row topic" title="topic_140">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0140">Protein targeting and localization (Protein sorting)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">36.16%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.60%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0245">Protein structural motif recognition (Protein structural feature identification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.08%</td>
</tr>

<tr class="row operation" title="operation_2436">
<td class="tp"><a href="http://edamontology.org/operation_2436">Functional enrichment (GO term enrichment)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.66%</td>
</tr>

<tr class="row operation" title="operation_2241">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2241">Transmembrane protein visualisation (Transmembrane protein rendering)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.72%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.90%</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="NA">Mascot Parser</a></h3>
<p>PDOM and MASCOT parser </p>
</td></tr>

<tr class="row topic" title="topic_3398">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3398">Bioengineering (Diagnostic markers)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.90%</td>
</tr>

<tr class="row topic" title="topic_3368">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3368">Biomaterials (Diagnostic markers)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.90%</td>
</tr>

<tr class="row topic" title="topic_3360">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3360">Biomarkers (Diagnostic markers)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.90%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_281">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0281" class="obsolete">Genetic marker identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.56%</td>
</tr>

<tr class="row operation" title="operation_444">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0444">Matrix/scaffold attachment site prediction (Identify matrix/scaffold attachment regions (MARs/SARs) in DNA sequences.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">3.27%</td>
</tr>

<tr class="row operation" title="operation_347">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0347" class="obsolete">Sequence database search (by motif or pattern)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">2.86%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_1478">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1478">PDBML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="row format" title="format_1580">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1580" class="obsolete">ProDom entry format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://proteomics.ethz.ch/muellelu/web/LukasReiter/Mayu/">Mayu</a></h3>
<p>estimates protein identification false discovery rates</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.08%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">24.38%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3649">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3649">Target-Decoy (Estimation of false discovery rate by comparison to search results with a database containing incorrect information.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">21.07%</td>
</tr>

<tr class="row operation" title="operation_245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0245">Protein structural motif recognition (Protein structural feature identification)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.29%</td>
</tr>

<tr class="row operation" title="operation_3648">
<td class="tp"><a href="http://edamontology.org/operation_3648">Validation of peptide-spectrum matches (Statistical estimation of false discovery rate from score distribution for peptide-spectrum-matches, following a peptide database search.)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">18.39%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://kirchnerlab.github.com/libmgf">ms2preproc</a></h3>
<p>ms2preproc - MGF and dta fragment spectrum preprocessing </p>
</td></tr>

<tr class="row topic" title="topic_654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0654">DNA</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.72%</td>
</tr>

<tr class="row topic" title="topic_2229">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2229">Cell biology (Cellular processes)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.00%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3219">Read pre-processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_2443">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2443" class="obsolete">Phylogenetic tree processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_3008">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3008">MAF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.72%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://sourceforge.net/project/showfiles.php?group_id=109078&package_id=169183">mres2x</a></h3>
<p>a tool to process MASCOT results </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_2229">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2229">Cell biology (Cellular processes)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">16.88%</td>
</tr>

<tr class="row topic" title="topic_3527">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3527" class="obsolete">Cellular process pathways</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (Processing)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata) (Tool information retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.78%</td>
</tr>

<tr class="row operation" title="operation_2407">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2407" class="obsolete">Annotation processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_3713">
<td class="tp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://code.google.com/p/ehu-bio/wiki/PAnalyzer">PAnalyzer</a></h3>
<p>rearranges Waters PLGS XML or mzIdentML peptide data into inferred proteins</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">39.58%</td>
</tr>

<tr class="row topic" title="topic_3366">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3366">Data integration and warehousing (Data integration)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_3572">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3572">Data quality management (Data integrity)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_227">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0227">Indexing (Data indexing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.13%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.06%</td>
</tr>

<tr class="row operation" title="operation_2412">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2412" class="obsolete">Data index processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.29%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_2332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2332">XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://tools.proteomecenter.org/PeptideProphet.php">PeptideProphet</a></h3>
<p>validation of peptide identifications by MS/MS and database searches </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">46.61%</td>
</tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.24%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.24%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">59.82%</td>
</tr>

<tr class="row operation" title="operation_3646">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3646">Peptide database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">57.66%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.79%</td>
</tr>

<tr class="row operation" title="operation_3648">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3648">Validation of peptide-spectrum matches</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file ("Raw" result file from Mascot database search.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="row format" title="format_1332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1332">FASTA search results format (Format of results of a sequence database search using FASTA.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.19%</td>
</tr>

<tr class="row format" title="format_2562">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2562" class="obsolete">Amino acid identifier format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.16%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/ProteinProphet.php">ProteinProphet</a></h3>
<p>validation of protein identifications by MS/MS and database searches </p>
</td></tr>

<tr class="row topic" title="topic_2271">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2271" class="obsolete">Structure database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.24%</td>
</tr>

<tr class="row topic" title="topic_163">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0163" class="obsolete">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.24%</td>
</tr>

<tr class="row topic" title="topic_3346">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3346" class="obsolete">Sequence search (Sequence database search)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">42.98%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2421">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2421">Database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">59.82%</td>
</tr>

<tr class="row operation" title="operation_343">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0343" class="obsolete">Transmembrane protein database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">48.16%</td>
</tr>

<tr class="row operation" title="operation_340">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0340" class="obsolete">Protein secondary database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.77%</td>
</tr>

<tr class="row operation" title="operation_3648">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3648">Validation of peptide-spectrum matches</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file ("Raw" result file from Mascot database search.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://code.google.com/p/xtandem-parser/">XTandem Parser</a></h3>
<p>Java-based parser for X!Tandem result files </p>
</td></tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.32%</td>
</tr>

<tr class="row topic" title="topic_219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0219">Data submission, annotation and curation (Database curation)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">10.13%</td>
</tr>

<tr class="row topic" title="topic_3038">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3038" class="obsolete">Structure databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting (File reformatting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (File processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="row operation" title="operation_3359">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3359">Splitting (File splitting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.36%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://pappso.inra.fr/bioinfo/xtandempipeline/">X!TandemPipeline</a></h3>
<p>filter, edit and group peptide/protein identifications from X!Tandem (and MASCOT)</p>
</td></tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.44%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">40.00%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.39%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3695">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3695">Filtering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.81%</td>
</tr>

<tr class="row operation" title="operation_3096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3096">Editing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_3711">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3711">X!Tandem XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="NA">APEX</a></h3>
<p>protein abundance estimate from LC-MS/MS data </p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.71%</td>
</tr>

<tr class="row topic" title="topic_2225">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2225" class="obsolete">Protein databases (Protein data resources)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">30.94%</td>
</tr>

<tr class="row topic" title="topic_3534">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3534">Protein binding sites</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.46%</td>
</tr>

<tr class="row operation" title="operation_537">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0537" class="obsolete">Protein structure assignment (from NMR data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.76%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">32.07%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/ASAPRatio.php">ASAPRatio</a></h3>
<p>statistical analysis of protein ratios from ICAT, cICAT or SILAC experiments</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.19%</td>
</tr>

<tr class="row topic" title="topic_2269">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2269">Statistics and probability (Statistics)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">37.50%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation (Statistical analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">50.80%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.19%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2406">Protein structure analysis (Structure analysis (protein))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.67%</td>
</tr>

<tr class="row operation" title="operation_3638">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3638">SILAC</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row format" title="format_3578">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3578">IDAT</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.93%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://omics.pnl.gov/software/DAnTE.php">DAnTE</a></h3>
<p>protein quantitation, statistical analysis and visualization</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.31%</td>
</tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row topic" title="topic_639">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0639" class="obsolete">Protein sequence analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation (Statistical analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">49.88%</td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.31%</td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">42.75%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3630">Quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row format" title="format_2062">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2062">Protein report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://sourceforge.net/projects/icplquant/">ICPLQuant</a></h3>
<p>tool for processing and analyzing ICPL LC-MALDI data </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.55%</td>
</tr>

<tr class="row operation" title="operation_3705">
<td class="tp"><a href="http://edamontology.org/operation_3705">Isotope-coded protein label (ICPL)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (Processing)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://sourceforge.net/projects/icplquant/">ICPL_ESIQuant</a></h3>
<p>tool for processing and analyzing ICPL LC-MS/MS (ESI) data </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.27%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">20.09%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row operation" title="operation_3705">
<td class="tp"><a href="http://edamontology.org/operation_3705">Isotope-coded protein label (ICPL)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (Processing)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.31%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://bioinformatics.cemm.oeaw.ac.at/isobar/">isobar</a></h3>
<p>quantitation of TMT and iTRAQ data and LaTeX report generation </p>
</td></tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.57%</td>
</tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation (Data rendering)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.89%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.71%</td>
</tr>

<tr class="row operation" title="operation_3480">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3480">Probabilistic data generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.80%</td>
</tr>

<tr class="row operation" title="operation_3639">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3641">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3641">TMT-tag</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1615">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1615" class="obsolete">GeneDB gene report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.18%</td>
</tr>

<tr class="row format" title="format_2329">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2329" class="obsolete">GeneCards gene report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.53%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://norwegian-proteomics-society.uio.no/isobariq/">IsobariQ</a></h3>
<p>quantitation of IPTL, iTRAQ and TMT-labeled peptides </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">40.50%</td>
</tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3135">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3135" class="obsolete">Signal or transit peptide</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.90%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">42.75%</td>
</tr>

<tr class="row operation" title="operation_3639">
<td class="tp"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_3641">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3641">TMT-tag</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1430">Phylip continuous quantitative characters</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_2037">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2037">Phylogenetic continuous quantitative character format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="row format" title="format_3248">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3248">mzQuantML (mzQuantML is the format for quantitation values associated with peptides, proteins and small molecules from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of quantitation software for proteomics.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.80%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/Libra.php">Libra</a></h3>
<p>analyzes 4- and 8-channel iTRAQ data </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.00%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.94%</td>
</tr>

<tr class="row topic" title="topic_3370">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3370">Analytical chemistry</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.46%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3639">
<td class="tp"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row operation" title="operation_3641">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3641">TMT-tag</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://pappso.inra.fr/bioinfo/masschroq/">MassChroQ</a></h3>
<p>XIC extraction and quantitation from LC-MS data </p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_3572">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3572">Data quality management (Data quality)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">17.14%</td>
</tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.43%</td>
</tr>

<tr class="row operation" title="operation_537">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0537" class="obsolete">Protein structure assignment (from NMR data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row operation" title="operation_3638">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3638">SILAC</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3639">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3641">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3641">TMT-tag</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3634">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3634">Label-free quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.maxquant.org/">MaxQuant</a></h3>
<p>quantitation from SILAC data from Thermo Orbitrap and FTICR</p>
</td></tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row topic" title="topic_3572">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3572">Data quality management (Data quality)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.08%</td>
</tr>

<tr class="row operation" title="operation_3638">
<td class="tp"><a href="http://edamontology.org/operation_3638">SILAC</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.16%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3712">Thermo RAW (Proprietary file format for mass spectrometry data from Thermo Scientific.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.32%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://mfpaq.sourceforge.net">MFPaQ</a></h3>
<p>Mascot file parsing and quantitation using ICAT and SILAC </p>
</td></tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (File management)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="row topic" title="topic_3519">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3519">PCR experiment (PCR experiments, e.g. quantitative real-time PCR.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">8.32%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3638">
<td class="tp"><a href="http://edamontology.org/operation_3638">SILAC</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">30.54%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0335">Formatting (File reformatting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="row operation" title="operation_3639">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.55%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.93%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (File format)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://msquant.sourceforge.net">MSQuant</a></h3>
<p>protein quantitation combining Mascot results and raw data from stable isotope labeling</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">38.86%</td>
</tr>

<tr class="row topic" title="topic_3539">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3539" class="obsolete">Protein domains</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row topic" title="topic_130">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0130">Protein folding, stability and design (Protein stability)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.81%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_273">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0273">Protein interaction raw data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.73%</td>
</tr>

<tr class="row operation" title="operation_3705">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3705">Isotope-coded protein label</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.32%</td>
</tr>

<tr class="row operation" title="operation_3715">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3715">Metabolic labeling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.84%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3630">Quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.91%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row format" title="format_1638">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1638">cel (Affymetrix probe raw data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.71%</td>
</tr>

<tr class="row format" title="format_3713">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3713">Mascot .dat file</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://sourceforge.net/projects/ms-spectre">MS-Spectre</a></h3>
<p>quantitiave analysis of multiple LC-MS(/MS) analyses in mzXML</p>
</td></tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.62%</td>
</tr>

<tr class="row topic" title="topic_81">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0081">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.62%</td>
</tr>

<tr class="row topic" title="topic_1775">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_1775">Function analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.62%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.12%</td>
</tr>

<tr class="row operation" title="operation_3204">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3204">Methylation analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.62%</td>
</tr>

<tr class="row operation" title="operation_3659">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3659">Regression analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.62%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3630">Quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row format" title="format_3158">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3158">PSI MI XML (MIF)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.80%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.msstats.org/">MSstats</a></h3>
<p>statistical tool for quantitative mass spectrometry-based proteomics </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_2269">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2269">Statistics and probability (Statistics)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">39.38%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation (Statistics)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">41.56%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td class="tp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row operation" title="operation_3658">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3658">Statistical inference</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.92%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_3681">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3681">mzTab (mzTab is a tab-delimited format for mass spectrometry-based proteomics and metabolomics results.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">18.23%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://ms.iis.sinica.edu.tw/Multi-Q/">Multi-Q</a></h3>
<p>tool for multiplexed iTRAQ-based quantitation </p>
</td></tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row topic" title="topic_3519">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3519">PCR experiment (PCR experiments, e.g. quantitative real-time PCR.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.29%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.53%</td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata) (Tool information retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.78%</td>
</tr>

<tr class="row operation" title="operation_3634">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3634">Label-free quantification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.09%</td>
</tr>

<tr class="row operation" title="operation_3639">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1430">Phylip continuous quantitative characters</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_2037">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2037">Phylogenetic continuous quantitative character format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_3281">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3281">A2M (The A2M format is used as the primary format for multiple alignments of protein or nucleic-acid sequences in the SAM suite of tools. It is a small modification of FASTA format for sequences and is compatible with most tools that read FASTA.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/muxQuant">muxQuant</a></h3>
<p>multiplexed quantitiave proteomics using differential stable isotope labeling </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3715">
<td class="tp"><a href="http://edamontology.org/operation_3715">Metabolic labeling</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.24%</td>
</tr>

<tr class="row operation" title="operation_3705">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3705">Isotope-coded protein label</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.33%</td>
</tr>

<tr class="row operation" title="operation_3640">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3640">18O labeling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_3635">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3635">Labeled quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.05%</td>
</tr>

<tr class="row format" title="format_3254">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3254">KRSS2 Syntax (This format is used in Protege 4.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">11.13%</td>
</tr>

<tr class="row format" title="format_3691">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3691">BEL</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.44%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="http://www.bioinfor.com/peaks/products/peaks-q-solution.html">PEAKS Q</a></h3>
<p>peptide/protein quantification by iTRAQ, ICAT, SILAC or label-free </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">38.57%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.86%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">31.38%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3634">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3634">Label-free quantification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.14%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.97%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td class="tp"><a href="http://edamontology.org/operation_3630">Quantification</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row format" title="format_3578">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3578">IDAT</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.ms-utils.org/pepXML2Excel.html">pepXML2Excel</a></h3>
<p>converts output from PeptideProphet to protein level information in Excel</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Protein informatics)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.72%</td>
</tr>

<tr class="row topic" title="topic_166">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0166">Protein structural motifs and surfaces (Protein 3D motifs)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.80%</td>
</tr>

<tr class="row topic" title="topic_595">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0595" class="obsolete">Protein classification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.13%</td>
</tr>

<tr class="row operation" title="operation_390">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0390">Protein peeling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_2467">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2467" class="obsolete">Data retrieval (protein annotation) (Protein information retrieval)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.73%</td>
</tr>

<tr class="row operation" title="operation_335">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3655">pepXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3468">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3468">xls</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="13">
<h3><a href="https://code.google.com/archive/p/prorata/">ProRata</a></h3>
<p>differential proteomics analysis using for various stable isotope labeling schemes </p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.52%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.52%</td>
</tr>

<tr class="row operation" title="operation_3223">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3223">Differential expression analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.56%</td>
</tr>

<tr class="row operation" title="operation_3677">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3677">Differential binding analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.56%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.90%</td>
</tr>

<tr class="row format" title="format_3254">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3254">KRSS2 Syntax (This format is used in Protege 4.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">9.41%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.55%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://compbio.cs.princeton.edu/pview/">PVIEW</a></h3>
<p>isotope labeled, label-free, and XIC-based quantitation for high resolution LC-ESI-MS/MS data </p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.24%</td>
</tr>

<tr class="row topic" title="topic_3572">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3572">Data quality management (Data quality)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">17.20%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.96%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3634">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3634">Label-free quantification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.48%</td>
</tr>

<tr class="row operation" title="operation_3635">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3635">Labeled quantification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.13%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.13%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.63%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.63%</td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://sourceforge.net/projects/protms">Quant</a></h3>
<p>MATLAB program for protein quantitation by iTRAQ </p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.50%</td>
</tr>

<tr class="row topic" title="topic_166">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0166">Protein structural motifs and surfaces (Protein 3D motifs)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.58%</td>
</tr>

<tr class="row topic" title="topic_595">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0595" class="obsolete">Protein classification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3639">
<td class="tp"><a href="http://edamontology.org/operation_3639">iTRAQ</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.50%</td>
</tr>

<tr class="row operation" title="operation_2997">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2997">Protein comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_3626">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3626">MAT (MATLAB file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.00%</td>
</tr>

<tr class="row format" title="format_1934">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1934">fitch program</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ncbi.nlm.nih.gov/pubmed/16906720">QUIL</a></h3>
<p>another program for relative quantitation using stable isotope labeling </p>
</td></tr>

<tr class="row topic" title="topic_3055">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3055">Quantitative genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Computer programming)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.84%</td>
</tr>

<tr class="row topic" title="topic_130">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0130">Protein folding, stability and design (Protein stability)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3630">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3630">Quantification (Quantitation)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row operation" title="operation_3715">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3715">Metabolic labeling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.24%</td>
</tr>

<tr class="row operation" title="operation_3635">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3635">Labeled quantification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1934">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1934">fitch program</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row format" title="format_1430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1430">Phylip continuous quantitative characters</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_2037">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2037">Phylogenetic continuous quantitative character format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://informatics.mayo.edu/svn/trunk/mprc/raams/index.html">RAAMS</a></h3>
<p>algorithm for interpreting O-16/O-18 differential proteomics data </p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Algorithms)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">38.57%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis (Protein data processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.63%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.55%</td>
</tr>

<tr class="row operation" title="operation_2405">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2405" class="obsolete">Protein interaction data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.26%</td>
</tr>

<tr class="row operation" title="operation_3640">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3640">18O labeling</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://fields.scripps.edu/downloadfile2.php?name=RelEx&filename=RelEx.zip&id=4">RelEx</a></h3>
<p>calculation of ion current ratios from LC-MS data (requires Xcalibur) </p>
</td></tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row topic" title="topic_3077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3077">Data acquisition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row operation" title="operation_3704">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3704">Ion counting (Ion current integration)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.67%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.66%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1925">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1925">codata</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.97%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.51%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.89%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://tools.proteomecenter.org/XPRESS.php">XPRESS</a></h3>
<p>calculates relative abundances from ICAT, cICAT, SILAC and other N-14/N-15 experiments </p>
</td></tr>

<tr class="row topic" title="topic_3656">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3656">Immunoprecipitation experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.16%</td>
</tr>

<tr class="row topic" title="topic_3516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3516">Genotyping experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.16%</td>
</tr>

<tr class="row topic" title="topic_3518">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3518">Microarray experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.16%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">40.91%</td>
</tr>

<tr class="row operation" title="operation_3638">
<td class="tp"><a href="http://edamontology.org/operation_3638">SILAC</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.27%</td>
</tr>

<tr class="row operation" title="operation_383">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0383" class="obsolete">Protein hydropathy calculation (from structure)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.17%</td>
</tr>

<tr class="row operation" title="operation_3715">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3715">Metabolic labeling</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3578">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3578">IDAT</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.49%</td>
</tr>

<tr class="row format" title="format_3167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3167">Experiment annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.61%</td>
</tr>

<tr class="row format" title="format_1928">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1928">Staden experiment format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.61%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://mass-spec.biologie.uni-osnabrueck.de/CC/cgi/form_digest.pl">CCdigest</a></h3>
<p>digest proteins with intact disulfide bridges</p>
</td></tr>

<tr class="row topic" title="topic_128">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0128">Protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.38%</td>
</tr>

<tr class="row topic" title="topic_3557">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3557">Protein interaction experiment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row topic" title="topic_3044">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3044" class="obsolete">Protein interaction networks</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2492">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2492">Protein interaction prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row operation" title="operation_2949">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2949">Protein interaction analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row operation" title="operation_2445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2445" class="obsolete">Protein interaction network processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.21%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://mass-spec.biologie.uni-osnabrueck.de/CC/cgi/form_find.pl">CCfind</a></h3>
<p>find peptides with disulfide bridges from ETD data</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.90%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.94%</td>
</tr>

<tr class="row operation" title="operation_3647">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3647">Blind peptide database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.06%</td>
</tr>

<tr class="row operation" title="operation_304">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0304">Metadata retrieval (Data retrieval (metadata))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.58%</td>
</tr>

<tr class="row operation" title="operation_3092">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3092">Protein feature detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.27%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="NA">MS2Pro</a></h3>
<p>assigns crosslinked or non-crosslinked fragments in whole protein MS/MS (top-down) spectra </p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">28.50%</td>
</tr>

<tr class="row topic" title="topic_150">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0150" class="obsolete">Protein design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_166">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0166">Protein structural motifs and surfaces (Protein 3D motifs)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.58%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_320">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0320">Protein structure assignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">24.17%</td>
</tr>

<tr class="row operation" title="operation_2929">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.78%</td>
</tr>

<tr class="row operation" title="operation_319">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0319">Protein secondary structure assignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="NA">MS2Assign</a></h3>
<p>assigns crosslinked or non-crosslinked peptides in MS/MS spectra </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">28.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.18%</td>
</tr>

<tr class="row operation" title="operation_2929">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2929">Protein fragment weight comparison (Peptide mass fingerprinting)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.66%</td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3651">MGF (Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">15.18%</td>
</tr>

<tr class="row format" title="format_3655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3655">pepXML (Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">13.31%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.31%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://patternlabforproteomics.org/sim-xl">SIM-XL</a></h3>
<p>Spectrum Identification Machine for Cross-Linked Peptides </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">40.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">31.50%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.67%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3631">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3631">Peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.00%</td>
</tr>

<tr class="row operation" title="operation_3643">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3643">Tag-based peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.67%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.94%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3092">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3092">Protein feature detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3684">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3684">PRIDE XML (PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">8.57%</td>
</tr>

<tr class="row format" title="format_2562">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2562" class="obsolete">Amino acid identifier format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.44%</td>
</tr>

<tr class="row format" title="format_3247">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3247">mzIdentML (mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.18%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.stavrox.com/">StavroX</a></h3>
<p>analyses peptide-peptide cross-links for protein-protein interaction or protein structure analysis </p>
</td></tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">65.25%</td>
</tr>

<tr class="row topic" title="topic_81">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0081">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">56.92%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.54%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2406">
<td class="tp"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">65.25%</td>
</tr>

<tr class="row operation" title="operation_2480">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2480">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">56.92%</td>
</tr>

<tr class="row operation" title="operation_2949">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2949">Protein interaction analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">49.11%</td>
</tr>

<tr class="row operation" title="operation_3092">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3092">Protein feature detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.65%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_2065">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2065">Protein structure report (quality evaluation) format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="https://goodlett.umaryland.edu/xcomb.php">xComb</a></h3>
<p>interogate protein-protein interactions and protein structure using cross-links FASTA databases </p>
</td></tr>

<tr class="row topic" title="topic_3526">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3526" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.40%</td>
</tr>

<tr class="row topic" title="topic_147">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0147" class="obsolete">Protein-protein interactions</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.40%</td>
</tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis (Protein structure)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">46.30%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_275">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0275" class="obsolete">Protein-protein interaction prediction (from protein structure)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.48%</td>
</tr>

<tr class="row operation" title="operation_474">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0474">Protein structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.24%</td>
</tr>

<tr class="row operation" title="operation_320">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0320">Protein structure assignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.24%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3092">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3092">Protein feature detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.47%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://tools.proteomecenter.org/XLink.php">XLink</a></h3>
<p>set of tools for analysis of data from crosslinked and digested protein complexes</p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.56%</td>
</tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.22%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2502">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2502">Protein analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">30.56%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3092">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3092">Protein feature detection</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_2607">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2607">completely unambiguous pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.48%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.28%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://www.ms-utils.org/autohd.html">AUTOHD</a></h3>
<p>analysis of H/D-exchange data using isotopic distributions </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">36.09%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.02%</td>
</tr>

<tr class="row topic" title="topic_3375">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3375">Pharmacokinetics and pharmacodynamics (Drug distribution)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.70%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.22%</td>
</tr>

<tr class="row operation" title="operation_3632">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.09%</td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.50%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_404">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0404">Protein hydrogen exchange rate calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.93%</td>
</tr>

<tr class="row format" title="format_3706">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3706">Biodiversity data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.71%</td>
</tr>

<tr class="row format" title="format_2210">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2210" class="obsolete">Strain data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.71%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="19">
<h3><a href="http://hdx.florida.scripps.edu/hdx_workbench/Home.html">Deuterator</a></h3>
<p>web based software platform for determining backbone amide H/D ratios </p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Amino acids)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.19%</td>
</tr>

<tr class="row topic" title="topic_2226">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2226" class="obsolete">Structure determination</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3185">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3185">Base-calling (Base calling)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.25%</td>
</tr>

<tr class="row operation" title="operation_479">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0479">Protein modelling (backbone)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.83%</td>
</tr>

<tr class="row operation" title="operation_287">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0287">Base position variability plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.75%</td>
</tr>

<tr class="row operation" title="operation_570">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0570">Structure visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3351">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3351">Molecular surface analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_404">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0404">Protein hydrogen exchange rate calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1633">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1633">PHD</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_3003">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3003">BED</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_3286">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3286">PED</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row format" title="format_3654">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3654">mzXML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="NA">DEX</a></h3>
<p>another program (similar to AUTOHD) for analysis of H/D-exchange spectra </p>
</td></tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row topic" title="topic_81">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0081">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row topic" title="topic_1775">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_1775">Function analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.94%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.75%</td>
</tr>

<tr class="row operation" title="operation_3204">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3204">Methylation analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1934">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1934">fitch program</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Exchange format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">13.92%</td>
</tr>

<tr class="row format" title="format_3653">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3653">pkl (Spectral data file similar to dta.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.11%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://hx2.med.upenn.edu/download.html">ExMS</a></h3>
<p>MATLAB program for analyzing H/D-exchange data </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.37%</td>
</tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases (Sequence data)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_92">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0092">Data visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.66%</td>
</tr>

<tr class="row operation" title="operation_228">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0228" class="obsolete">Data index analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.34%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_404">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0404">Protein hydrogen exchange rate calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row format" title="format_3626">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3626">MAT (MATLAB file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.81%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="19">
<h3><a href="http://hdx.florida.scripps.edu/hdx_workbench/Home.html">HDX WorkBench</a></h3>
<p>integrated software platform for HDX data analysis </p>
</td></tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">52.50%</td>
</tr>

<tr class="row topic" title="topic_3366">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3366">Data integration and warehousing (Data integration)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row topic" title="topic_3572">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3572">Data quality management (Data integrity)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">30.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3445">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3445">Diffraction data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_2495">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2495">Gene expression data analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.46%</td>
</tr>

<tr class="row operation" title="operation_570">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0570">Structure visualisation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3351">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3351">Molecular surface analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_404">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0404">Protein hydrogen exchange rate calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">15.83%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row format" title="format_3712">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3712">Thermo RAW</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="https://github.com/jtprince/hydrogen_bondifier">hydrogen_bondifier</a></h3>
<p>PyMOL script for retrieving hydrogen bond characteristics from protein structures</p>
</td></tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis (Protein structure)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">50.80%</td>
</tr>

<tr class="row topic" title="topic_172">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0172" class="obsolete">Protein structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row topic" title="topic_143">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0143" class="obsolete">Protein structure comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_275">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0275" class="obsolete">Protein-protein interaction prediction (from protein structure)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">38.19%</td>
</tr>

<tr class="row operation" title="operation_360">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0360">Structural similarity search (Structure retrieval by structure)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">34.31%</td>
</tr>

<tr class="row operation" title="operation_474">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0474">Protein structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_404">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0404">Protein hydrogen exchange rate calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row format" title="format_2065">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2065">Protein structure report (quality evaluation) format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row format" title="format_2077">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2077">Protein secondary structure format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.hxms.com/HXExpress/">HX-Express</a></h3>
<p>an Excel spreadsheet for analysis of H/D exchange mass spectra </p>
</td></tr>

<tr class="row topic" title="topic_217">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0217" class="obsolete">Literature analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row topic" title="topic_81">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0081">Structure analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row topic" title="topic_1775">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_1775">Function analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">39.01%</td>
</tr>

<tr class="row operation" title="operation_3627">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3627">Mass spectra calibration</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row operation" title="operation_2945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2945">Analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.94%</td>
</tr>

<tr class="row operation" title="operation_2406">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2406">Protein structure analysis</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_404">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0404">Protein hydrogen exchange rate calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Exchange format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.47%</td>
</tr>

<tr class="row format" title="format_3468">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3468">xls (Microsoft Excel spreadsheet format.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row format" title="format_1633">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1633">PHD</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.94%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://en.wikipedia.org/wiki/Bash_%28Unix_shell%29">bash one-liner</a></h3>
<p>bash one-liner</p>
</td></tr>

<tr class="row topic" title="topic_769">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0769">Workflows (Pipelines)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">7.63%</td>
</tr>

<tr class="row topic" title="topic_594">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0594" class="obsolete">Sequence classification (The classification of molecular sequences based on some measure of their similarity.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">4.02%</td>
</tr>

<tr class="row topic" title="topic_164">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0164" class="obsolete">Sequence clustering (The comparison and grouping together of molecular sequences on the basis of their similarities.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">4.02%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3185">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3185">Base-calling (Base calling)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.81%</td>
</tr>

<tr class="row operation" title="operation_287">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0287">Base position variability plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row operation" title="operation_521">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0521" class="obsolete">PCR primer design (based on gene structure)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.33%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2920">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2920">Alignment format (pair only)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="row format" title="format_1333">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1333">BLAST results</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.50%</td>
</tr>

<tr class="row format" title="format_3331">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3331">BLAST XML results format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.63%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://www.ms-utils.org/dig/dig.html">dig</a></h3>
<p>digests or fragments protein sequences </p>
</td></tr>

<tr class="row topic" title="topic_3043">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3043" class="obsolete">Protein sequences</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">65.63%</td>
</tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">46.88%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.53%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2516">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2516" class="obsolete">Protein sequence visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_2509">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2509">Protein sequence comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_366">
<td class="tp"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_2096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2096">unambiguous sequence</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row format" title="format_2171">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2171">Sequence cluster format (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.ms-utils.org/dig2/dig2.html">dig2</a></h3>
<p>updated version of dig</p>
</td></tr>

<tr class="row topic" title="topic_3577">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3577">Personalized medicine</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.93%</td>
</tr>

<tr class="row topic" title="topic_622">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0622">Genomics (Personal genomics)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">8.04%</td>
</tr>

<tr class="row topic" title="topic_749">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0749">Transcription factors and regulatory sites (Terminators)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">4.29%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_249">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0249">Protein geometry calculation (Torsion angle calculation)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">6.43%</td>
</tr>

<tr class="row operation" title="operation_1826">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1826" class="obsolete">Full torsion angle calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row operation" title="operation_1827">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_1827" class="obsolete">Cysteine torsion angle calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.25%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3005">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3005">WIG</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_1942">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1942">ig</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_3578">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3578">IDAT</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://indigestionapp.com">InDigestion</a></h3>
<p>iOS app for in-silico digestion and more</p>
</td></tr>

<tr class="row topic" title="topic_663">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0663" class="obsolete">tRNA (One or more transfer RNA (tRNA) sequences.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.22%</td>
</tr>

<tr class="row topic" title="topic_660">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0660" class="obsolete">rRNA (One or more ribosomal RNA (rRNA) sequences.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.22%</td>
</tr>

<tr class="row topic" title="topic_2275">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2275">Molecular modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0365">Nucleic acid restriction digest</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row operation" title="operation_2932">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2932" class="obsolete">Hopp and Woods plotting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.17%</td>
</tr>

<tr class="row operation" title="operation_3704">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3704">Ion counting</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.75%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3693">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3693">AGP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.00%</td>
</tr>

<tr class="row format" title="format_3728">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3728">LocARNA PP</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">5.83%</td>
</tr>

<tr class="row format" title="format_3726">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3726">PMML ( One or more mining models can be contained in a PMML document.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">5.60%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://prospector.ucsf.edu/prospector/cgi-bin/msform.cgi?form=msdigest">MS-Digest</a></h3>
<p>digests a protein sequence</p>
</td></tr>

<tr class="row topic" title="topic_3043">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3043" class="obsolete">Protein sequences</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score good">72.92%</td>
</tr>

<tr class="row topic" title="topic_741">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0741" class="obsolete">Protein sequence alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_639">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0639" class="obsolete">Protein sequence analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_366">
<td class="tp"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row operation" title="operation_2479">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2479">Protein sequence analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row operation" title="operation_2509">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2509">Protein sequence comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.33%</td>
</tr>

<tr class="row format" title="format_2096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2096">unambiguous sequence</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_2171">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2171">Sequence cluster format (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2571">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2571">Raw sequence format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://pchopper.lifesci.dundee.ac.uk">PChopper</a></h3>
<p>multiple-enzyme ''in silico'' digester for experimental design</p>
</td></tr>

<tr class="row topic" title="topic_3678">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3678">Experimental design and studies (Experimental design)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">58.19%</td>
</tr>

<tr class="row topic" title="topic_150">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0150" class="obsolete">Protein design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_209">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0209">Medicinal chemistry (Drug design)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.94%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2430">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2430">Design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row operation" title="operation_332">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0332" class="obsolete">Immunogen design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row operation" title="operation_2419">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2419">Primer and probe design</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1641">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1641">affymetrix-exp (Affymetrix experimental conditions data file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">8.71%</td>
</tr>

<tr class="row format" title="format_3725">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3725">SBOL (SBOL introduces a standardized format for the electronic exchange of information on the structural and functional aspects of biological designs.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">5.31%</td>
</tr>

<tr class="row format" title="format_3602">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3602">pgm (It is designed to be extremely easy to learn and write programs for.)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">4.58%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://www.expasy.ch/tools/peptidecutter">PeptideCutter</a></h3>
<p>another web based protein digestion applet</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row topic" title="topic_166">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0166">Protein structural motifs and surfaces (Protein 3D motifs)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.79%</td>
</tr>

<tr class="row topic" title="topic_3518">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3518">Microarray experiment (Reverse phase protein array)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">16.88%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2997">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2997">Protein comparison</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row operation" title="operation_477">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0477">Protein modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row operation" title="operation_302">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0302">Protein threading</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2330">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2330">Textual format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://omics.pnl.gov/software/protein-digestion-simulator">Protein Digestion Simulator Basic</a></h3>
<p>calculates theoretical enzymatic peptides and AMTs</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">27.00%</td>
</tr>

<tr class="row topic" title="topic_771">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0771" class="obsolete">Theoretical biology</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.75%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row operation" title="operation_334">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0334">Enzyme kinetics calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.05%</td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3013">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3013">axt</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row format" title="format_2323">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2323" class="obsolete">ENZYME enzyme report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.84%</td>
</tr>

<tr class="row format" title="format_1514">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1514" class="obsolete">KEGG ENZYME enzyme report format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.50%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="https://github.com/jgmeyerucsd/ProteomeDigestSim">ProteomeDigestSim</a></h3>
<p>R classes for prediction of proteome coverage based on iterative digestion by MED-FASP or similar </p>
</td></tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_644">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0644" class="obsolete">Proteome</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_2278">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2278" class="obsolete">Transmembrane protein prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.35%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2423">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2423">Prediction and recognition (Prediction)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">24.75%</td>
</tr>

<tr class="row operation" title="operation_440">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0440">Promoter prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="row operation" title="operation_269">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0269">Transmembrane protein prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.35%</td>
</tr>

<tr class="row operation" title="operation_366">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0366">Protein sequence cleavage</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.25%</td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.79%</td>
</tr>

<tr class="row format" title="format_3097">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3097">Protein domain classification format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.50%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.maldi-msi.org/index.php?option=com_content&view=article&id=87&Itemid=48">Axima2Analyze</a></h3>
<p>converts imaging data from the Axima formats to the Analyze format used in BioMap</p>
</td></tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_3365">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3365">Data architecture, analysis and design (Data analysis)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">19.29%</td>
</tr>

<tr class="row topic" title="topic_3571">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3571">Data governance</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.04%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3383">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3383">Biological imaging</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_335">
<td class="tp"><a href="http://edamontology.org/operation_0335">Formatting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">32.36%</td>
</tr>

<tr class="row operation" title="operation_3434">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3434">Conversion</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row operation" title="operation_3443">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3443">Image analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.43%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3547">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3547">Image format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.86%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">34.60%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">33.75%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.amolf.nl/download/datacubeexplorer/">Datacube Explorer</a></h3>
<p>vendor-neutral visualization program for mass spectrometry imaging data (including imzML) </p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.83%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3383">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3383">Biological imaging</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_337">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0337">Visualisation (Visualization)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">26.39%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">18.47%</td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3682">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3682">imzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">41.67%</td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.24%</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://www.maldi-msi.org/index.php?option=com_content&view=article&id=315&Itemid=102">MSImageView</a></h3>
<p>generate images from WIFF data and export to imzML</p>
</td></tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.71%</td>
</tr>

<tr class="row topic" title="topic_3303">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3303">Medicine (General medicine)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.70%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3383">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3383">Biological imaging</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_543">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0543">Phylogenetic tree construction (from polymorphism data) (Phylogenetic tree generation (from polymorphism data))</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">24.03%</td>
</tr>

<tr class="row operation" title="operation_3480">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3480">Probabilistic data generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.81%</td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3682">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3682">imzML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_3710">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3710">WIFF format (wiff)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.14%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www4.ncsu.edu/~dcmuddim/msireader.html">MSiReader</a></h3>
<p>vendor-neutral MATLAB program to view analyze mass spectrometry imaging data </p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.83%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">41.25%</td>
</tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3383">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3383">Biological imaging</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.53%</td>
</tr>

<tr class="row operation" title="operation_3443">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3443">Image analysis</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.83%</td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">39.24%</td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat (Affymetrix image data file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">22.17%</td>
</tr>

<tr class="row format" title="format_3652">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3652">dta</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://lababi.bioprocess.org/index.php/lababi-software/79-openmzxy">OpenMZxy</a></h3>
<p>open mass spectrometry imaging control software </p>
</td></tr>

<tr class="row topic" title="topic_134">
<td class="tp"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">50.00%</td>
</tr>

<tr class="row topic" title="topic_3520">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3520">Proteomics experiment (Mass spectrometry)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3383">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3383">Biological imaging</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3694">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3694">Mass spectrum visualisation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="row operation" title="operation_3214">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3214">Spectral analysis (Mass spectrum analysis)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">20.78%</td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row format" title="format_3682">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3682">imzML (imzML is a data format for mass spectrometry imaging data. NB.: See comment.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_3547">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3547">Image format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://code.google.com/p/brain-isotopic-distribution/">BRAIN</a></h3>
<p>algorithm for calculating aggregated isotope distributions and corresponding center-masses (R package)</p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Algorithms)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">37.50%</td>
</tr>

<tr class="row topic" title="topic_3176">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3176">DNA packaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.22%</td>
</tr>

<tr class="row topic" title="topic_3375">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3375">Pharmacokinetics and pharmacodynamics (Drug distribution)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">13.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">44.44%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="row operation" title="operation_3436">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3436">Aggregation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.78%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2074">Dirichlet distribution format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.11%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.11%</td>
</tr>

<tr class="row format" title="format_1392">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1392">DIALIGN format (Format of multiple sequences aligned by DIALIGN package.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">6.86%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://cran.r-project.org/web/packages/enviPat/index.html">enviPat</a></h3>
<p>batch fine structure, profile and centroid calculation with resolution settings provided</p>
</td></tr>

<tr class="row topic" title="topic_167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0167" class="obsolete">Structural (3D) profiles (Structural profiles)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">44.86%</td>
</tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis (Protein structure)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">21.11%</td>
</tr>

<tr class="row topic" title="topic_172">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0172" class="obsolete">Protein structure prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.27%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_2411">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2411" class="obsolete">Structural profile processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.26%</td>
</tr>

<tr class="row operation" title="operation_297">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0297">3D profile generation (Structural profile construction)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">32.55%</td>
</tr>

<tr class="row operation" title="operation_299">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0299">3D profile-to-3D profile alignment (Structural profile alignment)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">32.55%</td>
</tr>

<tr class="row operation" title="operation_3632">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1990">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1990">match</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1419">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1419">Sequence-MEME profile alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.04%</td>
</tr>

<tr class="row format" title="format_2069">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2069">Sequence profile format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.04%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://proteome.gs.washington.edu/software/IDCalc">IDCalc</a></h3>
<p>isotope distribution and theoretical spectrum calculation with GUI</p>
</td></tr>

<tr class="row topic" title="topic_771">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0771" class="obsolete">Theoretical biology</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_3307">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3307">Computational biology (Theoretical biology)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_3375">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3375">Pharmacokinetics and pharmacodynamics (Drug distribution)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">57.29%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2074">Dirichlet distribution format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_1349">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1349">HMMER Dirichlet prior (Dirichlet distribution HMMER format.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">5.21%</td>
</tr>

<tr class="row format" title="format_1350">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1350">MEME Dirichlet prior (Dirichlet distribution MEME format.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">5.21%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://blog.binfalse.de/software/iso2l/">iso2l</a></h3>
<p>calculates the isotopic distribution of a chemical formula or a amino acid chain</p>
</td></tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Amino acids)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row topic" title="topic_613">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0613" class="obsolete">Peptides and amino acids</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">34.38%</td>
</tr>

<tr class="row topic" title="topic_3369">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3369">Chemical biology</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.27%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row operation" title="operation_348">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0348" class="obsolete">Sequence database search (by amino acid composition)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2035">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2035">Chemical formula format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.95%</td>
</tr>

<tr class="row format" title="format_2562">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2562" class="obsolete">Amino acid identifier format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row format" title="format_2017">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2017">Amino acid index format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.46%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_1197">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1197">InChI</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/isotop.html">isotop</a></h3>
<p>MATLAB code for calculating the isotopic distribution of a molecule (''e''.''g''. insulin)</p>
</td></tr>

<tr class="row topic" title="topic_2839">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2839" class="obsolete">Molecules</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.07%</td>
</tr>

<tr class="row topic" title="topic_655">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0655" class="obsolete">Coding RNA</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">53.14%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row operation" title="operation_387">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0387">Molecular surface calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.82%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3626">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3626">MAT (MATLAB file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.33%</td>
</tr>

<tr class="row format" title="format_3681">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3681">mzTab</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row format" title="format_2074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2074">Dirichlet distribution format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/isotop.r">isotop</a></h3>
<p>the same program in R</p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Computer programming)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">34.53%</td>
</tr>

<tr class="row topic" title="topic_3277">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3277">Sample collections (samples)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">4.76%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation (Function (programming))</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">32.71%</td>
</tr>

<tr class="row operation" title="operation_338">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0338">Sequence database search (This excludes direct retrieval methods (e.g. the dbfetch program).)</a></td>
<td>&nbsp;</td>
<td class="type">comment</td>
<td class="score bad">12.96%</td>
</tr>

<tr class="row operation" title="operation_528">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0528" class="obsolete">SAGE data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row operation" title="operation_3632">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1934">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1934">fitch program</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">36.34%</td>
</tr>

<tr class="row format" title="format_2573">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2573">SAM</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_3554">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3554">R file format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.kombyonyx.com/isotopes/">Isotope calculator</a></h3>
<p>console application for calculating isotope distributions based on Yergey's method</p>
</td></tr>

<tr class="row topic" title="topic_3375">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3375">Pharmacokinetics and pharmacodynamics (Drug distribution)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">14.06%</td>
</tr>

<tr class="row topic" title="topic_3569">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3569">Applied mathematics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">10.42%</td>
</tr>

<tr class="row topic" title="topic_89">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0089">Ontology and terminology (Applied ontology)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">47.27%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation (Computational method)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">21.09%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2074">Dirichlet distribution format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.46%</td>
</tr>

<tr class="row format" title="format_3157">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3157">EBI Application Result XML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.38%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">7.81%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://www.ms-utils.org/isotop_fs.html">isotop_fs</a></h3>
<p>MATLAB code for calculating isotopic fine structure</p>
</td></tr>

<tr class="row topic" title="topic_114">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0114">Gene structure</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_697">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0697" class="obsolete">RNA structure</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_2814">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2814">Protein structure analysis (Protein structure)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row operation" title="operation_360">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0360">Structural similarity search (Structure retrieval by structure)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">25.73%</td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.52%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3626">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3626">MAT (MATLAB file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">23.75%</td>
</tr>

<tr class="row format" title="format_3681">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3681">mzTab</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">14.58%</td>
</tr>

<tr class="row format" title="format_2033">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2033">Tertiary structure format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="15">
<h3><a href="http://www.ms-utils.org/isotope.py">isotope.py</a></h3>
<p>the same program in Python</p>
</td></tr>

<tr class="row topic" title="topic_3372">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3372">Software engineering (Computer programming)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.79%</td>
</tr>

<tr class="row topic" title="topic_783">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0783">Pathogens</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row topic" title="topic_3277">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3277">Sample collections (samples)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">9.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_4">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0004">Operation (Function (programming))</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row operation" title="operation_3280">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3280">Named entity recognition</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row operation" title="operation_528">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0528" class="obsolete">SAGE data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row operation" title="operation_3632">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2573">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2573">SAM</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1934">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1934">fitch program</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.83%</td>
</tr>

<tr class="row format" title="format_1945">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1945">mase format (Mase program sequence format.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">7.29%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="https://github.com/kirchnerlab/libipaca">libipaca</a></h3>
<p>Isotopic PAttern CAlculation library (MIT license)</p>
</td></tr>

<tr class="row topic" title="topic_3341">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3341">Clone library</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row topic" title="topic_3343">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3343">Compound libraries and screening (Compound library)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row topic" title="topic_3444">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3444">MRI (MRT)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.24%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">31.25%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row operation" title="operation_3629">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3629">Deisotoping</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">21.88%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1357">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1357">EMBOSS sequence pattern</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_3626">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3626">MAT</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">9.72%</td>
</tr>

<tr class="row format" title="format_3158">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3158">PSI MI XML (MIF) (MIF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">9.24%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="14">
<h3><a href="http://omics.pnl.gov/software/molecular-weight-calculator">Molecular Weight Calculator</a></h3>
<p>calculates molecular masses and isotopic distributions</p>
</td></tr>

<tr class="row topic" title="topic_3375">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3375">Pharmacokinetics and pharmacodynamics (Drug distribution)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">23.63%</td>
</tr>

<tr class="row topic" title="topic_2839">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2839" class="obsolete">Molecules</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.61%</td>
</tr>

<tr class="row topic" title="topic_2275">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2275">Molecular modelling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.50%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">53.33%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.00%</td>
</tr>

<tr class="row operation" title="operation_387">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0387">Molecular surface calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.29%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2074">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2074">Dirichlet distribution format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">13.33%</td>
</tr>

<tr class="row format" title="format_2054">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2054">Protein interaction format (Molecular interaction format)</a></td>
<td>&nbsp;</td>
<td class="type">broad_synonym</td>
<td class="score bad">12.00%</td>
</tr>

<tr class="row format" title="format_3245">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3245">Mass spectrometry data format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://prospector.ucsf.edu/prospector/cgi-bin/msform.cgi?form=msisotope">MS-Isotope</a></h3>
<p>calculates the isotopic distribution from a peptide sequence</p>
</td></tr>

<tr class="row topic" title="topic_3168">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3168">Sequencing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequences)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">41.56%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3524">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3524">Simulation experiment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3632">
<td class="tp"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.56%</td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_401">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0401" class="obsolete">Protein hydropathy calculation (from sequence)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">35.21%</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_2096">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2096">unambiguous sequence</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row format" title="format_1967">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1967">DAS format (das sequence format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">19.79%</td>
</tr>

<tr class="row format" title="format_3701">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3701">sequin</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">18.75%</td>
</tr>

<tr class="row format" title="format_1957">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1957">raw</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://pappso.inra.fr/bioinfo/pdm/">Database Manager</a></h3>
<p>sequence database manager, including random database generation</p>
</td></tr>

<tr class="row topic" title="topic_3041">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3041" class="obsolete">Sequence databases</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">59.82%</td>
</tr>

<tr class="row topic" title="topic_80">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0080">Sequence analysis (Sequence databases)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">53.84%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td class="tp"><a href="http://edamontology.org/topic_3489">Database management</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">51.79%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_230">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0230">Sequence generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">48.21%</td>
</tr>

<tr class="row operation" title="operation_338">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0338">Sequence database search</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">45.24%</td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.86%</td>
</tr>

<tr class="row operation" title="operation_3649">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3649">Target-Decoy</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.57%</td>
</tr>

<tr class="row format" title="format_2066">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2066">Database hits (sequence) format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.66%</td>
</tr>

<tr class="row format" title="format_2078">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2078">Sequence range format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.74%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="16">
<h3><a href="http://www.matrixscience.com/help/decoy_help.html">decoy.pl</a></h3>
<p>decoy database generator from Matrix Science</p>
</td></tr>

<tr class="row topic" title="topic_2816">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2816" class="obsolete">Gene resources (Gene database)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">31.67%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row topic" title="topic_3390">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3390">Nutritional science</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row operation" title="operation_542">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0542">Phylogenetic tree generation (from gene frequencies)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row operation" title="operation_540">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0540">Phylogenetic tree generation (from molecular sequences)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.79%</td>
</tr>

<tr class="row operation" title="operation_3649">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3649">Target-Decoy</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.44%</td>
</tr>

<tr class="row format" title="format_3033">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3033">Matrix format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">16.67%</td>
</tr>

<tr class="row format" title="format_2329">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2329" class="obsolete">GeneCards gene report format (Format of a report on a gene from the GeneCards database.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">16.51%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="18">
<h3><a href="http://www.ms-utils.org/make_random.html">make_random</a></h3>
<p>generate random database through Markov process</p>
</td></tr>

<tr class="row topic" title="topic_2816">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2816" class="obsolete">Gene resources (Gene database)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">34.83%</td>
</tr>

<tr class="row topic" title="topic_3489">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3489">Database management (Databases)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">27.71%</td>
</tr>

<tr class="row topic" title="topic_3053">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3053">Genetics</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_2269">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_2269">Statistics and probability</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3429">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3429">Generation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">43.75%</td>
</tr>

<tr class="row operation" title="operation_2409">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2409">Data handling (Processing)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score medium">39.38%</td>
</tr>

<tr class="row operation" title="operation_2412">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2412" class="obsolete">Data index processing (Database index processing)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">29.69%</td>
</tr>

<tr class="row operation" title="operation_3649">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3649">Target-Decoy</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2238">Statistical calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3011">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3011">genePred</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">29.17%</td>
</tr>

<tr class="row format" title="format_2031">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2031">Gene annotation format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_2175">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2175" class="obsolete">Gene cluster format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">11.25%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.ms-utils.org/compareMS2.html">compareMS2</a></h3>
<p>calculates overlap between two LC-MS/MS datasets in MGF </p>
</td></tr>

<tr class="row topic" title="topic_3518">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3518">Microarray experiment (Two channel microarray)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">10.31%</td>
</tr>

<tr class="row topic" title="topic_2839">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2839" class="obsolete">Molecules</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">6.03%</td>
</tr>

<tr class="row topic" title="topic_3473">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3473">Data mining (The discovery of patterns in large data sets and the extraction and trasnsformation of those patterns into a useful format.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">4.40%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3293">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3293">Phylogenetics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row operation" title="operation_2238">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2238">Statistical calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.63%</td>
</tr>

<tr class="row operation" title="operation_3278">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3278">Document similarity calculation (Calculate similarity between 2 or more documents.)</a></td>
<td>&nbsp;</td>
<td class="type">definition</td>
<td class="score bad">11.66%</td>
</tr>

<tr class="row operation" title="operation_2424">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2424">Comparison</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_3651">
<td class="tp"><a href="http://edamontology.org/format_3651">MGF</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row format" title="format_3702">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_3702">MSF</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.10%</td>
</tr>

<tr class="row format" title="format_1947">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1947">GCG MSF (MSF)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">14.35%</td>
</tr>

<tr class="row format" title="format_1912">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1912">Nexus format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="http://www.quadtechassociates.com/freeware.html">Graph Extract</a></h3>
<p>tool for digitizing data from images, similar to the classic Data Grabber and WebPlotDigitizer</p>
</td></tr>

<tr class="row topic" title="topic_3382">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_3382">Imaging</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">27.50%</td>
</tr>

<tr class="row topic" title="topic_2807">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2807" class="obsolete">Tool topic</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.50%</td>
</tr>

<tr class="row topic" title="topic_2230">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_2230" class="obsolete">Classification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.64%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_218">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0218">Text mining</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_515">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0515" class="obsolete">Data retrieval (tool metadata)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">20.00%</td>
</tr>

<tr class="row operation" title="operation_535">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0535">Protein secondary structure assignment (from CD data)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.97%</td>
</tr>

<tr class="row operation" title="operation_530">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0530" class="obsolete">SBS data processing</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">19.88%</td>
</tr>

<tr class="row operation" title="operation_3441">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3441">Plotting</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1637">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1637">dat (Affymetrix image data file format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">17.22%</td>
</tr>

<tr class="row format" title="format_1915">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1915">Format (Data format)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score bad">16.62%</td>
</tr>

<tr class="row format" title="format_3547">
<td class="tp"><a href="http://edamontology.org/format_3547">Image format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">15.00%</td>
</tr>

<tr class="row format" title="format_3475">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3475">TSV</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="17">
<h3><a href="https://github.com/princelab/mspire-simulator">mspire-simulator</a></h3>
<p>simulates peptide retention times and chromatographic profiles</p>
</td></tr>

<tr class="row topic" title="topic_167">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0167" class="obsolete">Structural (3D) profiles</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">26.62%</td>
</tr>

<tr class="row topic" title="topic_154">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0154">Small molecules (Peptides)</a></td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">26.25%</td>
</tr>

<tr class="row topic" title="topic_3524">
<td class="tp"><a href="http://edamontology.org/topic_3524">Simulation experiment</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row topic" title="topic_121">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0121">Proteomics</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3633">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3633">Retention times prediction</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">37.50%</td>
</tr>

<tr class="row operation" title="operation_316">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0316" class="obsolete">Functional profiling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_3200">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3200">Community profiling</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">25.00%</td>
</tr>

<tr class="row operation" title="operation_2426">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_2426">Modelling and simulation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1419">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1419">Sequence-MEME profile alignment</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_2069">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2069">Sequence profile format</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">12.50%</td>
</tr>

<tr class="row format" title="format_2585">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_2585">SBML</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">8.75%</td>
</tr>

<tr class="row format" title="format_1929">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_1929">FASTA</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_3244">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_3244">mzML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep"><td colspan="6">&nbsp;</td></tr>

<tr class="query"><td rowspan="20">
<h3><a href="http://www.protpi.ch">Prot pi</a></h3>
<p>online calculation (simulation) of many physicochemical parameters of peptides/proteins</p>
</td></tr>

<tr class="row topic" title="topic_78">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0078">Protein analysis (Proteins)</a></td>
<td>&nbsp;</td>
<td class="type">exact_synonym</td>
<td class="score medium">40.08%</td>
</tr>

<tr class="row topic" title="topic_767">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/topic_0767" class="obsolete">Protein and peptide identification</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.65%</td>
</tr>

<tr class="row topic" title="topic_121">
<td class="tp"><a href="http://edamontology.org/topic_0121">Proteomics (Protein and peptide identification)</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="type">narrow_synonym</td>
<td class="score bad">25.78%</td>
</tr>

<tr class="row topic" title="topic_134">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_0134" class="obsolete">Mass spectrometry</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3541">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3541" class="obsolete">Protein post-translational modifications</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row topic" title="topic_3537">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/topic_3537" class="obsolete">Protein chemical modifications</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row operation" title="operation_3438">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_3438">Calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">28.13%</td>
</tr>

<tr class="row operation" title="operation_378">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_0378" class="obsolete">Sequence composition calculation (protein)</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row operation" title="operation_2574">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/operation_2574">Protein hydropathy calculation</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">22.92%</td>
</tr>

<tr class="row operation" title="operation_400">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0400">Protein pH-dependent property calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_250">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_0250">Protein property calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row operation" title="operation_3632">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/operation_3632">Isotopic distributions calculation</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="sep-branch">
<td colspan="5"></td>
</tr>

<tr class="row format" title="format_1208">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1208">protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score medium">42.19%</td>
</tr>

<tr class="row format" title="format_1219">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1219">pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">23.44%</td>
</tr>

<tr class="row format" title="format_1218">
<td>&nbsp;</td>
<td class="fp"><a href="http://edamontology.org/format_1218">unambiguous pure protein</a></td>
<td>&nbsp;</td>
<td class="type">label</td>
<td class="score bad">17.19%</td>
</tr>

<tr class="row format" title="format_2571">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2571">Raw sequence format</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

<tr class="row format" title="format_2331">
<td>&nbsp;</td>
<td>&nbsp;</td>
<td class="fn"><a href="http://edamontology.org/format_2331">HTML</a></td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>

</tbody>

<tfoot>
<tr>
<td>236</td>
<td>163</td>
<td>1961</td>
<td>670</td>
<td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tfoot>

</table>

</body>
</html>
